%\documentclass[b5paper]{report}
%\documentclass[a4crop]{ntnuthesis}
\documentclass[]{ntnuthesis}

% A new package for bibliography
% Use biber for bibtex and pdflatex instead of latex
%\usepackage[backend=biber, url=true]{biblatex}
%\usepackage[backend=biber, citestyle=numeric-comp, doi=false, isbn=false,
%url=false, sort=false]{biblatex}
\usepackage[backend=biber, citestyle=numeric-comp, doi=false, isbn=false,
url=false, sorting=none]{biblatex}
% new from biblatex
\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}

% Support for unicode. Should be unnecessary.
%\usepackage[utf8]{inputenc}
%\usepackage[T1]{fontenc}

\usepackage{amsmath,amsfonts,amssymb,amsthm,booktabs,array,mathtools}
% consider package mhchem for typesetting chemical formulas
\usepackage{verbatim}

% Proper space and font for integral differential term
\newcommand{\dd}{\; \mathrm{d}} 
% Shorcut for ODEs with proper font
\newcommand{\diff}[2]{\frac{\mathrm{d} #1}{\mathrm{d} #2}}
% Shortcut for PDEs with proper font (shortcut: PDB)
\newcommand{\pdiff}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\pdiffn}[3]{\frac{\partial^{#3} #1}{\partial #2^{#3}}}

\usepackage{rotating} % make your own prime symbol
\usepackage{adjustbox} % make your own prime symbol
% make primes
\newcommand*{\p}{\adjustbox{raise=0.4ex, right=0.56ex}{\scalebox{0.7}{$'$}}\;}
\newcommand*{\pp}{\adjustbox{raise=0.4ex, right=0.56ex}{\scalebox{0.7}{$'$}}\;\,}

% Absolute value and norm commands.
% Read the mathtools.pdf to fix these!
\providecommand{\abs}[1]{\lvert#1\rvert} 
\providecommand{\norm}[1]{\lVert#1\rVert}

% Set the depth of section numbering
\setcounter{secnumdepth}{0}

\title{Regulation of gene expression by untranslated regions}
\author{J\o rgen Skancke}
%\subtitle{The unlikely story of how you got scammed into buying this thesis and
%loved every part of it}
\degreetype{philosophiae doctor}
\faculty{The Faculty of Natural Sciences and Technology}
\department{Department of Chemical Engineering}

\date{August 2012}

\begin{document} 

\frontmatter

%\maketitle

\chapter*{Acknowledgements}
\input{subsections/acknowledgements.tex}

% Uncomment this when you get to this stage
%\pagenumbering{roman}
%\tableofcontents
%\listoffigures
%\listoftables

\mainmatter

%\chapter{Introduction}

%\input{subsections/gene_expression_general.tex}

%\section{Transcription initiation in bacteria}
%\subsection{Biochemical background}
%\input{subsections/transcription_background.tex}
%\subsection{Computational models}
%\input{subsections/transcription_computation.tex}

%\section{Translation initiation in bacteria}
%\input{subsections/translation_background.tex}

%\section{Transcription termination in eukaryotes}
%\input{subsections/polyadenylation.tex}
%\section{High throughput RNA sequencing}
%\input{subsections/rna_seq.tex}

%\part{Regulation of transcription and translation initiation in the 5 prime
%untranslated region}

 %Abortive initiation
%\chapter{Abortive transcription initiation is regulated by the free energy of
%the scrunched DNA bubble and translocation bias}

% Infa2b expression
%\chapter{Project with Veronika -- bioinformatic approach to acheive expression
%of infa2b in \textit{E coli}}
%\input{subsections/celB_pelB_introduction.tex}

% Friedrieke paper -> moving a structure
%\chapter{Increasing translation initiation by clearing the ribosome binding
%site of a strong secondary structure}
%\input{subsections/friederieke_summary.tex}

% Polyadenylation
%\part{Using RNA-seq for indentification of \textit{de novo} sites of
%polyadenylation}

\chapter{Contibution to ENCODE paper}
\input{subsections/polyA_results.tex}

%\endmatter

\appendix
\chapter{Utail -- finding poly(A) sites from short read data}
\input{subsections/utail.tex}

\printbibliography

\end{document}
