%\bibliography{/home/jorgsk/phdproject/bibtex/jorgsk}

The cell is considered to be the unit of life. A single celled organism manages
by itself to absorb nutrients from its environment and uses those nutrients to
navigate and to replicate. On the surface, life seems to be about
consuming nutrients and replicating. It is possible to stop with this
understanding of life and move on. But if we want to understand how the cell
uses these external metabolites like sugars and fatty acids to drive cell
division and cellular locomotion -- and we do -- then we must look under the
surface to the cell interior. Inside the cell we see a crowded milieu of DNA,
RNA, and protein molecules in constant collision and reaction. But we also see
a system to the chaos, and that system begins with DNA being the blueprint or
instruction set from which all cellular programs are executed. Ultimately, the
DNA blueprint is used to produce RNA and protein molecules that through
interactions on the micro scale are able to cause macro scale changes such as
cell division or motility.

Genes are the discrete parts of DNA molecules which are copied into RNA, and gene
expression is another term for RNA and protein production. Gene expression
consist broadly of two stages: transcription, in which an RNA copy is made
from DNA, and translation, in which a protein is created based on
the genetic code in the RNA sequence. For some genes, for example transfer RNA
(tRNA) and ribosomal RNA (rRNA), gene expression stops after transcription.
Here, the RNA molecule is on its own competent to perform a structural or
enzymatic function. In this work however, we will deal with RNA that is
translated to protein, so called messenger RNA (mRNA).

A basic property of gene expression is that it is carefully regulated. The
bacterial cell \textit{Eschericia coli} contains roughly 4000 genes, but at a
given time only a subset of these are expressed. Which genes are expressed at a
given time depends on the environmental condition the cell finds itself in,
Examples of these are nutrient availability, temperature, salt concentrations,
signals from other cells, and toxins; each of these input signals will have
unique or partially overlapping gene expression responses honed over the
course of evolution.

In this work we will not study how any of the mentioned environmental
conditions affect the pattern of gene expression. Instead, we will study gene
expression itself at several different stages. Specifically, we study i) the
initiation of transcription, ii) the termination of transcription, and iii) the
initiation of translation. The work on transcription and translation initiation
has been done with bacterial (prokaryotic) cells, while the work on
transcription termination has been done with human (eukaryotic) cells. It is
estimated that bacterial and eukaryotic cells have evolved independently for
around two billion years \cite{vellai_origin_1999}. Yet, the key players in
this thesis, the RNA polymerase and the ribosome, have remained structurally
and functionally conserved throughout that time.

In the following chapter we will review those stages of transcription and
translation that are relevant for the work at hand. The reviews will cover both
the biological and computational/theoretical background needed to read the
thesis as a whole.
