%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}

%What results should you present? Anything more than was presented in the paper?

%The figure with the compartments and poly(A)-.

%Shows two things: 1) there is poly(A) signal in poly(A)- extract. explained by
%errors in screening and by that some poly(A) sequences are short after library
%treatment. 2) intronic polyadenylation in the nucleus overrepresented. Makes
%sense if there is nuclear degradation of intronic poly(A) which has been
%recently shown.

%Then the figure with overlap to show how much you get?

%Then the figure that shows the flattening out of the getting of new
%transcripts.

%Discuss each of the tree figures and that's it.

%Maybe make a table with some of the numbers from the statistics you output.

%Then cite the paper.

\subsection{The dataset}
The datasets used in this study was generated by the ENCODE
(\textbf{Enc}yclopedia \textbf{O}f \textbf{D}NA \textbf{E}lements) consortium
and are available from http://hgdownload-test.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeCshlLongRnaSeq/

Table \ref{tab:Datasets} shows the cell lines and compartments used. In total,
data from 12 human cell lines were used. Each cell line is was sequenced for
the cytoplasmic and nuclear compartments as well as the whole cell, and most
datasets contained a biological replicated. In total, 23 datasets were from
whole cell extracts, 11 were from cytoplasmic extracts, and 12 were from
nuclear extracts. For each cell line and compartment, datasets are available
both for the poly(A)+ and poly(A)- fraction of the RNA pool. This brings the
total to 92 RNA-seq datasets. Each dataset has been generated with Illumina
paired-ended sequencing with a read-length of 75 basepairs.

\begin{table}
	\centering
	\begin{tabular}{cccc}
	  Cell line & Whole Cell & Cytoplasm & Nucleus \\
	  \midrule
	  GM12878 & 2 & 2 & 2 \\
	  K562 & 2 & 2 & 2 \\
	  HeLa-S3 & 2 & 2 & 2 \\
	  HUVEC & 2 & 2 & 2 \\
	  HEPG2 & 2 & 2 & 2 \\
	  H1Hesc & 1 & 1 & 1 \\
	  Nhek & 2 & 0 & 1 \\
	  MCF7 & 2 & 0 & 0 \\
	  AG04450 & 2 & 0 & 0 \\
	  HSMM & 2 & 0 & 0 \\
	  NHLF & 2 & 0 & 0 \\
	  A549 & 2 & 0 & 0 \\
	\end{tabular}
	\caption{Number of replicates of the datasets from the ENCODE consortium}
	\label{tab:Datasets}
\end{table}

The RNA library for this protocal was not especially targeted toward the 3' end
of the RNAs, but since poly(T) primers have been used for amplification and
cDNA creation, it is possible to find polyadenylated RNA in these data. What
the data lacks in specificity for the poly(A) sites, it makes up for it in
quantity, with more than 90 datasets from 12 cell line.

\subsection{The short RNA mapper}
The short read mapper used in this work is the GEM mapper CIT webpage. The GEM
mapper is developed in the group of Roderic Guigo and has been used regularly
in the group, although it has not been published yet.

\subsection{Merging and screening of poly(A) clusters}
In the results presented here, the poly(A) clusters for all datasets were
merged. We accepted as a poly(A) site a cluster with support from 3 or more
reads.

\section{Results}
We ran Utail on RNA-seq data that was part of the ENCODE consortium. Parts of
the analysis was published in XXX. Here we outline our contribution to the
paper, including the parts of the analysis which was not included in the final
paper.

There is good material in the reports.

You are still not sure: how close to the paper should you present your results?
You should probably ask Martin about this tomorrow. Now, go to the introduction
for the other stuff.

\subsection{k}
