% This is just for letting me cite stuff.
%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}

% What to include? Background on transcription in eukaryotes, polyadenylation,
% and rna-seq.

%\subsection{Terminology}
%\begin{itemize}
	%\item Read -- sequence fragment from a sequencing machine
	%\item RNA-seq -- high throughput RNA sequencing
	%\item PAS -- Polyadenylation signal, for example AAUAAA
	%\item Poly(A) cluster -- site in the genome where polyadenylated reads
		%cluster together
%\end{itemize}

\subsection{Eukaryotic transcription termination and polyadenylation}

\subsubsection{Transcription and transcript processing in mammals}
In eukaryotes there are several RNA polymerases, each one responsible for
transcribing different classes of genes. The most abundant polymerase is 
RNAP II and transcribes mostly protein coding genes. RNAP II is larger than its
bacterial counterpart, with 12 subunits in mammals, but the core subunits are
structurally and functionally conserved compared to bacterial RNAP
\cite{ebright_rna_2000}.

As in bacteria, transcription in eukaryotes begins at promoter sites, and
transcription initiation involves the same steps of open bubble formating and
abortive cycling, although many more assisting transcription factors are
involved \cite{wade_transition_2008}. Transcription elongation by RNAP II also
involves pausing and backtracking, but pausing on eukaryotic DNA is more
complicated due to the presence of chromatin and its methylation or acetylation
states \cite{sims_elongation_2004}. Transcription termination is markedly
different between eukaryotes and prokaryotes and will be covered more
extensively below.

In contrast to bacterial transcripts, mRNA in mammalian cells needs
extensive processing before it is capable of being translated by ribosomes.
Therefore, mRNA in eukaryotes is called pre-mRNA until it has gone through the
necessary processing steps. There are essentially three main events that make
up mRNA processing: 5\p cap addition, splicing, and 3\p cleavage and
polyadenylation.

The 5\p cap is a guanine nucleotide variant that is added to the 5\p nucleotide
of a transcript as it emerges from the RNA exit channel of RNAP II. The 5\p
cap presumably protects against 5\p exonucleases and aids in the splicing of
the 5\p intron.

Splicing is the removal of parts of pre-mRNA called introns. Introns are
non-coding parts of genes that actually make up most of gene sequences in
mammals. By splicing out the introns and joining the surrounding exons, only
the protein coding parts of the gene ends up in the final mRNA.

Cleavage and polyadenylation mark the 3\p end of the pre-mRNA. At designated
sites, the nascent transcript is cleaved in two while still being transcribed
by RNAP II. The 3\p end of the pre-mRNA is then catalytically extended with a
tail of around 250 adenosione nucleotides; this is generally the last step of
mRNA processing and marks the transition from pre-mRNA to mRNA. Below, we will
review in more detail the process of cleavage and polyadenylation.

\subsubsection{Cleavage and polyadenylation marks the 3' end of messenger RNAs
and promotes transcription termination}
Most protein coding genes contain a 3\p untranslated region (3\p
UTR) which may contain several sites where cleavage and polyadenylation can
occur. These sites are identified by the polyadenylation signal (PAS), which is
a highly conserved AAUAAA signal sequence or a close variant like AUUAAA. The
protein CPSF (cleavage/polyadenylation specificity factor) binds to the PAS and
with the help of other factors cleaves pre-mRNA 10-30 nucleotides downstream
the PAS. As soon as pre-mRNA has been cleaved, a poly(A)-polymerase will add
the poly(A) tail and the tail will be covered by poly(A) binding protein (PAB).

While the mRNA is now created, RNAP II is still transcribing downstream. One
characterized method of transcription termination is by a 5\p to 3\p
exonuclease called XRN2. In what is called the ``torpedo model'', this
exonuclease degrades the still-transcribed RNA until it reaches RNAP II and
causes transcription termination \cite{kuehner_unravelling_2011}.

\subsubsection{Usage of alternative polyadenylation sites}
% 1) For making different 3 UTR
Each 3\p UTR region of a gene may have several alternative sites where cleavage
and polyadenylation may happen. By chosing one site or the other, the length of
the 3\p UTR in the mRNA will be different. This can affect the
downstream fate of the mRNA as the 3\p UTR region contains many regulatory
elements, for example for transport and degradation, see
\cite{lutz_alternative_2011} for a recent review. Genome-wide changes in the
pattern of usage of polyadenylation sites has been shown during cell
proliferation \cite{sandberg_proliferating_2008} and for development
\cite{hilgers_neural-specific_2011, ji_progressive_2009}. This shows that the
selective usage of polyadenylation sites is part of several cellular programs.

There are also examples of cleavage and polyadenylation sites inside introns.
The use of such a site would leave out downstream exons, changing the protein
coding content of the mRNA. A well known example is that the membrane bound or
secreted versions of immunoglobin in B cells differ depending on whether
cleavage and polyadenylation occurs in an intron or not.

\subsubsection{High throughput RNA sequencing and genome-wide studies of
polyadenylation}

From the early 90s onward, more and more human expressed sequence tags (ESTs)
and cDNA became available. The increasing amount of sequence data facilitated
for the first time large scale analysis of 3\p UTRs and polyadenylation sites.
However, these studies were limited from the fact that the ESTs came from
different cell lines grown under different experimental conditions. Further,
the EST data was biased toward genes that were found interesting enough to
sequence individually. Such studies were nontheless successful in determining
the frequency of occurrance of the different PAS \cite{beaudoing_patterns_2000}
and that over half of human genes employ alternative polyadenylation and that
sites of alternative polyadenylation is conserved between humans and mice.
\cite{tian_large-scale_2005}.

With the advance of second generation sequencing technologies, sequence data
was no longer scarce. These technologies were fast used to study the
polyadenylation landscape for a single cells and tissues. These experiments
confirmed what had been discovered earlier by single-mRNA studies and EST
analysis; that AAUAAA is the canonical polyadenylation signal and that single
genes can be represented with multiple sites of polyadenylation. However, they
also brought new insight that could only be obtained with the genome wide
scale. Changes during development and proliferation were already mentioned, and
in addition a trend was found for overall shortening of 3'UTR lengths in for
cancer cells \cite{mayr_widespread_2009}. Most of these results were
reached with early studies and must be repeated to have their validity
ascertained. For example, a recent study did not find evidence for 3'UTR
shortening in cancer cells \cite{fu_differential_2011}, indicating that the
results from early studies should be reproduced.

\subsubsection{Preparing a cDNA library for RNA-sequencing}
The work presented in this thesis and most other high-throughput studies of
polyadenylation rely on sequencing data made with Illumina sequencing
technologies. Here we briefly mention the relevant stages of the RNA-seq
experiment.

To obtain good sequencing results, fragmented DNA in large concentrations is
required. To sequence RNA, therefore, it is necessary to both amplify the
cellular RNA extract and to convert the RNA to cDNA by reverse transcription.
Both RNA amplification and cDNA conversion can introduce biases depending on
the choices of methods and the order in which they are performed
\cite{wang_rna-seq:_2009}. From the point of view of polyadenylation, it is
necessary that the poly(A) tail is preserved during cDNA library preparation.
This requires the use of poly(T) primers both for making the cDNA library and
for PCR amplification.

After cDNA preparation and several rounds of PCR amplification, most poly(A)
tails will be reduced in length from the full 250 nt. This is because the
poly(T) primers will bind randomly in the poly(A) tail, something which will
gradually reduce the length of the tail.

\subsubsection{Sequencing errors and biases}
There are several errors and biases that creep into sequencing that must be
taken into account during the downstream computational analysis of the output.

An example of an error is that sequencing performance degrades during
sequencing increases over single-nucleotide stretches. This implies that when
sequencing the poly(A) sequence, other nucleotides than just A will be
reported. This means that when looking for poly(A) stretches in the output as
signatures of poly(A) sites, some variation from the mono-A sequence must be
expected.

An example of a bias is that the sequencing reads that cover poly(A) sites will
almost invariably be from the reverse-transcribed strand of the mRNA. This is
because sequencing, like transcription, happens in the 5'-> 3' direction, and
only the ends of the cDNA fragments are sequenced. Therefore, in practise, the
poly(A) stretches in the sequence output will, at least for Illumina sequencing
where the fragment length is more than twice the read length, result in
poly(T) output from the 5'-> 3' end. From now on, we will in general refer to
the poly(A/T) tail only as the poly(A) tail, even if the sequence in the reads
will often be poly(T).

\section{Methods - Utail}
We wanted to obtain information about sites of polyadenylation for RNA-seq
experiments so that we could rapidly analyze the polyadenylation pattern for
any RNA-seq experiment. To do that, we developed the tool \textit{Utail} (from
Untranslated poly(A) tail). If an RNA library has been prepared in a way that
ensures that the sites of polyadenylation will be included, Utail can use the
RNA-seq data from that library to locate and quantify the polyadenylation sites
and provide ready-to-use output.

\subsection{Utail pipeline overview}
The RNA-seq data is processed in a series of steps to go from raw data to
polyadenylation sites. The minimum requirements for Utail to run is mapped
RNA-seq reads. Presently, the reads must be provided either in the output
format of the GEM mapper CIT or as bed-files (see XXX for input
specifications). Although not necessary, the Utail output will be improved if a
genome annotation (in Ensemble or Gencode format) is supplied. The version of
the reference genome for the annotation must be the same as was used to map the
reads to in the first place.

Utail takes the mapped RNA-seq data through the following pipeline (each point
is elaborated on below):
\begin{enumerate}
	\item (Only if annotation was supplied) Create a list of all possible
		3'UTR regions and polyadenylation sites from the genome annotation
	\item Filter all reads and retain only the unmappable reads
	\item Keep only the unmappable reads with leading poly(T) or trailing
		poly(A) stretches
	\item Remove the poly(T) or poly(A) stretch from the read and remap the
		remaining part to the genome -- keep the uniquely mapping ones
	\item Determine the site of polyadenylation depending on poly(A/T) and which
		strand the read originated from
	\item Cluster together all reads that map within 40 nt of each other
	\item For each cluster, get the genomic sequence up to 60 nt downstream
		from the polyadenylation site. Note any PAS and discard cluster if a
		genomic poly(A) region is found at the site of the cleaved poly(A)
		stretch.
	\item For each poly(A) cluster, write to the output
\end{enumerate}
See Figure X for a graphical representation of the pipeline.

\textbf{1.)} By providing an annotation of the transcripts from the genome
re-map the read back to the genome. If the read maps uniquely (with up to 2
mismatches), consider this the 3' end of the mapping a putative poly(A) site.

\textbf{5.)} Even though the sequencing protocol is not strand-specific, we can
use the information from the poly(A) versus poly(T) stretch to find out if the
read has been reverse transcribed or not. For example, if the read maps to the
+ strand with a poly(A) tail, the read originated from the + strand. If the
read maps to the + strand with a poly(T) tail, the read originated from the -
strand. This allows the strand-specific determination of poly(A) sites.

\textbf{6.)} Since the exact site of cleavage and polyadenylation is stochastic,
we must cluster the different poly(A) sites. The average of this clustering is
reported as the cleavage and polyadenylation site for all reads in the cluster.

\textbf{7.)} The sequence surrounding the putative poly(A) site is searched for
one of the several reported PAS sequences as well as for a genomic poly(A)
stretch that could explain the poly(A) stretch thought to be a result of
polyadenylation. in
question, the set of already-annotated polyadenylation sites can be
constructed. If this step is included, it will be included in the output
whether a discovered polyadenylation site has already been annotated or not.

\textbf{2.)} With default settings, short-read mappers will not be able to map
to the genome a read-fragment containing the non-genomic poly(A)
stretch. Therefore, we are only interested in those reads which were initially
unmapped. The output format of mappers usually contains a flag which says if
the read was mapped or not.

\textbf{3.)} Among the unmapped reads, we are only interested in the ones which
contain a stretch of poly(A) at the 3' end or poly(T) at the 5' end. A minimum
of 5 poly(A/T) or 5 poly(A/T) in 7 was used.

\textbf{4.)} Once the poly(A) tail has been removed, we use the GEM mapper to
re-map the read back to the genome. If the read maps uniquely (with up to 2
mismatches), consider this the 3' end of the mapping a putative poly(A) site.

\textbf{5.)} Even though the sequencing protocol is not strand-specific, we can
use the information from the poly(A) versus poly(T) stretch to find out if the
read has been reverse transcribed or not. For example, if the read maps to the
+ strand with a poly(A) tail, the read originated from the + strand. If the
read maps to the + strand with a poly(T) tail, the read originated from the -
strand. This allows the strand-specific determination of poly(A) sites.

\textbf{6.)} Since the exact site of cleavage and polyadenylation is stochastic,
we must cluster the different poly(A) sites. The average of this clustering is
reported as the cleavage and polyadenylation site for all reads in the cluster.

\textbf{7.)} The sequence surrounding the putative poly(A) site is searched for
one of the several reported PAS sequences as well as for a genomic poly(A)
stretch that could explain the poly(A) stretch thought to be a result of
polyadenylation.

\textbf{8.)} The output for each cluster contains information about the
chromosome, strand, number of reads in the cluster, if the site has been
annotated, and if there is a PAS withing 40 nucleotides and the distance to
this PAS.

\subsection{Requirements and repository}
Utail can be obtained from its Git repository git@github.com:jorgsk/Utail.git.
In runs in a Linux/Unix environment and has the following requirements:
\begin{itemize}
	\item Python 2.6 ++
	\item BEDTools, a set of programs for manipulating .bed files
		\cite{quinlan_bedtools:_2010}
\end{itemize}
In addition, the following dependencies come bundled with Utail:
\begin{itemize}
	\item pyFasta, a Python module by Brent Pedersen for rapid extraction of
		genomic sequences from a reference genome
	\item bedGraphToBigWig, a program that converts bedGraph files to bigWig
		files, obtained from the ucsc.edu website.
\end{itemize}
Before running the program, modify the settings in he $UTR_SETTINGS$ file to your
preferences. Here you provide links to the RNA-seq data and the annotation and
set various settings. Run using ``python utail.py''; after completion the
output will be in the automatically created ``output'' folder.

\subsection{The dataset}
The datasets used in this study was generated by the ENCODE
(\textbf{Enc}yclopedia \textbf{O}f \textbf{D}NA \textbf{E}lements) consortium
and are available from http://hgdownload-test.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeCshlLongRnaSeq/

Table \ref{tab:Datasets} shows the cell lines and compartments used. In total,
data from 12 human cell lines were used. 23 datasets were from whole cell
extracts, 11 were from cytoplasmic extracts, and 12 were from nuclear extracts.
For each cell line and compartment, datasets are available both for the
poly(A)+ and poly(A)- fraction of the RNA pool). This brings the total to 92
RNA-seq datasets. Each dataset has been generated with Illumina paired-ended
sequencing with a read-length of 75 basepairs.
\begin{table}
	\centering
	\begin{tabular}{cccc}
	  Cell line & Whole Cell & Cytoplasm & Nucleus \\
	  \midrule
	  GM12878 & 2 & 2 & 2 \\
	  K562 & 2 & 2 & 2 \\
	  HeLa-S3 & 2 & 2 & 2 \\
	  HUVEC & 2 & 2 & 2 \\
	  HEPG2 & 2 & 2 & 2 \\
	  H1Hesc & 1 & 1 & 1 \\
	  Nhek & 2 & 0 & 1 \\
	  MCF7 & 2 & 0 & 0 \\
	  AG04450 & 2 & 0 & 0 \\
	  HSMM & 2 & 0 & 0 \\
	  NHLF & 2 & 0 & 0 \\
	  A549 & 2 & 0 & 0 \\
	\end{tabular}
	\caption{Number of replicates of the datasets from the ENCODE consortium}
	\label{tab:Datasets}
\end{table}

The RNA library for this protocal was not especially targeted toward the 3' end
of the RNAs, but since poly(T) primers have been used for amplification and
cDNA creation, it is possible to find polyadenylated RNA in these data. What
the data lacks in specificity for the poly(A) sites, it makes up for it in
quantity, with more than 90 datasets from 12 cell line.

\subsection{The short RNA mapper}
The short read mapper used in this work is the GEM mapper CIT webpage. The GEM
mapper is developed in the group of Roderic Guigo and has been used regularly
in the group the last years, although it has not been published yet.

\subsection{Merging and screening of poly(A) clusters}
In the results presented here, the poly(A) clusters for all datasets were
merged. We accepted as a poly(A) site a cluster with support from 3 or more
reads.

\section{Results}
We ran Utail on RNA-seq data that was part of the ENCODE consortium. Parts of
the analysis was published in XXX. Here we outline our contribution
to the paper, including the parts of the analysis which was not included in the
final paper.

There is good material in the reports.

You are still not sure: how close to the paper should you present your results?
You should probably ask Martin about this tomorrow. Now, go to the introduction
for the other stuff.

\subsection{k}


\end{document}


