% This is just for letting me cite stuff.
%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
\subsubsection{Overview of transcription in eukaryotes}
In this section we will review the process of transcription termination in in
mammalian cells, although much of the material is applicable to eukaryotes in
general. Since the previous sections were about transcription and translation
in bacteria, we will often make the contrast between transcription in
eukaryotes and in bacteria.

In eukaryotes there are several types of RNA polymerases, each one responsible
for transcribing different classes of genes. The polymerase which transcribes
protein coding genes is called RNAP II. With 12 subunits in mammals, RNAP II is
larger than its bacterial counterpart, but the core subunits are structurally
and functionally conserved compared to bacterial RNAP \cite{ebright_rna_2000}.

As in bacteria, transcription in eukaryotes begins when RNAP is recruited by
transcription factors to promoter sites. In eukaryotes there is no equivalent
to the bacterial sigma factor system, and there are more diversified methods
for recruiting RNAP to promoters. Transcription initiation in eukaryotes
involves the same steps of open bubble formating and abortive cycling as in
bacteria, although additional assisting transcription factors are involved in
this process compared to bacterial cells \cite{wade_transition_2008}.
Transcription elongation by RNAP also involves pausing and backtracking, but
pausing on eukaryotic DNA is more complicated due to the presence of
nucleosomes and chromatin \cite{sims_elongation_2004}.

Transcription termination on the other hand is markedly different between
eukaryotes and bacteria. In bacteria, the position where RNAP stops
transcribing also marks the genomic position of the 3\p end of the mRNA. In
eukaryotes however, transcription termination does not co-occur with mRNA
release and 3\p end definition. Instead, mRNA is released by a process of
cleavage and polyadenylation several hundred nucleotides before RNAP terminates
transcription. Cleavage and polyadenylation will be covered below in more
detail.

\subsubsection{mRNA processing is necessary for transport to the cytoplasm and
for translation}
In contrast to bacterial transcripts, eukaryotic mRNA must undergo extensive
processing in the nucleus before it is capable of being transported to the
cytoplasm for translation by ribosomes. Therefore, mRNA in eukaryotes is called
pre-mRNA until it has gone through the necessary processing steps. There are
essentially three main events that make up pre-mRNA processing: 5\p cap
addition, splicing, and 3\p cleavage and polyadenylation. Cleavage and
polyadenylation has already been covered, but here we briefly review 5\p
capping and splicing.

The 5\p cap is a guanine nucleotide variant that is added to the 5\p nucleotide
of a transcript shortly after it emerges from the RNA exit channel of RNAP II.
The 5\p cap presumably protects against 5\p exonucleases (RNA degrading protein
that attacks the 5\p end) and is necessary for the proper export of the mRNA
from the nucleus.

Splicing is the removal of introns. Eukaryotic protein coding genes are made up
of genetic regions called introns and exons. Introns are the non-coding parts
of pre-mRNA, and actually make up most of gene sequences in mammals, while
exons contain the sequences which are protein coding. By splicing out the
introns and joining the surrounding exons, only the exons of pre-mRNA make up
the final mRNA which is transported to the cytoplasm.

Several proteins have shared roles across the different pre-mRNA processing
steps. In other words, there is cross-talk between the processing pathways. For
example, the presence of the 5\p cap increases the efficiency of the excision
of the 5\p proximal intron and increases the efficiency of polyadenylation.
Polyadenylation on the other hand increases the efficiency of the excision of
the 3\p terminal intron \cite{proudfoot_integrating_2002}. A recently
discovered example of cross talk during mRNA processing is that the U1
ribonucleoprotein, previously known for its role in splicing, prevents
premature cleavage and polyadenylation at cryptic polyadenylation sites
\cite{kaida_u1_2010}.

If any of the processing steps are inefficiently executed or not executed at
all, the pre-mRNA transcript will be targeted for degradation either in the
nucleus itself or in the cytoplasm after transport \cite{doma_rna_2007}. This
quality control step reduces the risk that errors during transcription and
pre-mRNA processing will result in nonfunctional or possibly harmful protein
products.

\subsubsection{Usage of alternative polyadenylation sites}
% 1) For making different 3 UTR
Each 3\p UTR region of a gene often has several alternative sites where
cleavage and polyadenylation may happen \cite{tian_large-scale_2005}. Depending
on which of these sites are used for cleavage and polyadenylation, the length
of the 3\p UTR in the final mRNA will be different (a polyadenylation site
close to the beginning of the 3\p UTR will result in a short 3\p UTR while a
polyadenylation site far from the beginning of the 3\p UTR will result in a
long 3\p UTR). The choice of polyadenylation site can affect the downstream
fate of the mRNA, since the 3\p UTR region often contains regulatory elements.
A short 3\p UTR will in general contain fewer regulatory sequences than a long
3\p UTR. One of the best characterized examples of regulation in the 3\p UTR is
regulation by microRNA. microRNA are short RNA of around 20 nucleotides in
length that perform regulatory roles by basepairing with other RNA molecules.
When microRNA binds the 3\p UTR region of an mRNA in the cytoplasm, they
generally decrease expression from the mRNA they bind to. For a long time it
was unclear whether microRNA binding in the 3\p UTR decreased expression by
inducing transcript degradation or simply by blocking translation. Recently, it
was established that at least in mammals microRNA in the 3\p UTR decrease
expression by causing the transcript to be degraded
\cite{huntzinger_gene_2011}.

MicroRNA are known to regulate a host of metabolic processes and human diseases
through binding inthe 3\p UTR \cite{huang_biological_2010}. Therefore, the
choice of where to cleave and polyadenylate a transcript can have wide-spanning
consequences.

There are also examples of cleavage and polyadenylation in sites outside the
3\p UTR. The most prominent of these are sites within introns. The use of an
intronic cleavage and polyadenylation site would cause any downstream exons to
be left out of the transcript. In turn, this will result in a shorter peptide
sequence upon translation. Thus, the site of cleavage and polyadenylation can
also modulate the protein coding content of the mRNA. A well known example is
the case of the immunoglobin protein in B cells. Depending on whether a
polyadenylation site in an intron is used or not, the membrane bound or the
secreted version of the immunoglobin protein is made
\cite{peterson_regulated_1989}.

\subsubsection{Transient polyadenylation unrelated to mRNA processing}
In the last decade it has become clear that there is polyadenylation of RNA in
eukaryotic cells that is unrelated to mRNA 3\p processing. First, it was
discovered that poly(A) tails were added to aberrant transcripts in the nucleus
of yeast \cite{wyers_cryptic_2005}. The protein complex responsible for this
polyadenylation was given the name TRAMP, and transcripts polyadenylated in
this way were found to be targeted for degradation in the nucleus
\cite{lacava_rna_2005, wyers_cryptic_2005}. This was a surprising and important
discovery, as previously degradation-related polyadenylation was only known
from bacteria. The discovery prompted the suggestion that
degradation-associated polyadenylation by TRAMP has been conserved from
bacteria in the nucleus from the chimeric origin of the eukaryotic cell
\cite{lacava_rna_2005}. Later, degradation-related polyadenylation was found in
the nucleus of mammalian cells too, and eventually even in the cytoplasm of
human cells \cite{slomovic_polyadenylation_2006, slomovic_addition_2010}. In
summary, there is an emergent role for degradation-related polyadenylation of
some RNA species in eukaryotes; a mechanism that was previously only thought to
exist in bacteria. In contrast, the poly(A) tail formed on pre-mRNA protects
the mRNA against degradation. One possibility why the novel form of poly(A)
tail does not confer stability is that this tail is too short for the PAB to
bind \cite{lacava_rna_2005}, although the exact length of these poly(A) tails
was not determined in these studies.

\subsubsection{Genome-wide studies of polyadenylation}
The study of sites of polyadenylation across the genome has occurred in three
stages in the last 12 years, with each stage resting on a different type of
technology. The first wave used cDNA and EST sequence data (explained below)
obtained by laborious Sanger sequencing. The second wave used microarray and
SAGE technologies, and the third wave used the RNA-seq next generation
sequencing technology. We will review key results obtained in these three
stages chronologically.

From the early 90s onward, more and more human expressed sequence tags (ESTs)
became available. (An EST is the result of Sanger sequencing a part of cloned
RNA which has been isolated from a cell sample.) The increasing amount of
sequence data facilitated for the first time large scale analysis of 3\p UTRs
and polyadenylation sites.  The polyadenylation site of an EST is found by
identifying a poly(A) or poly(T) extremity which does not correspond to a
genomic sequence, trimming that extremity, and matching the remainder to
template mRNA or genomic sequence \cite{beaudoing_patterns_2000,
tian_large-scale_2005}.

These early genome-wide studies were successful in determining i) the frequency
of occurrence of the different PAS variants \cite{beaudoing_patterns_2000}, ii)
that over half of human genes employ alternative polyadenylation
\cite{tian_large-scale_2005}, iii) that sites of alternative polyadenylation
are evolutionary conserved between humans and mice
\cite{tian_large-scale_2005}, and vi) that there is extensive polyadenylation
in intronic polyadenylation sites \cite{tian_widespread_2007}.

However, EST data limits the type of questions that can be investigated. First,
EST data was in low quantity, due to the expense and time needed for Sanger
sequencing. Thus, the only practical way to compare alternative polyadenylation
on a genome-wide scale was to include EST data from different experiments,
often resulting in a mix of data from different cell lines and tissues. Our
literature review revealed no studies with \textit{de novo} EST sequencing for
the purpose of studying polyadenylation; all studies used EST sequences from
databases. Another limitation if EST data is that it is biased toward protein
coding genes that were found interesting enough to sequence individually. Thus
many classes of polyadenylated RNA, such as long noncoding RNA, were possibly
missed by these studies. Finally, although EST data may be used to give a
quantitative profile of gene expression, the output data is often normalized so
that the quantitative profile is lost. It therefore difficult to compare
expression values across genes with EST data, although some approaches have
been developed for this purpose \cite{liu_quantitative_2006}.

As the microarray technology matured and more full-length genomes became
available, microarrays, often in combination with EST data, were used to study
3\p UTR length variation and polyadenylation. Due to the low price and high
speed of microarrays, it was possible to use several microarray experiments in
the same study to compare the usage of alternative 3\p ends. To test for
variation in 3\p UTR length, microarray probes corresponding to the normal or
extended 3\p UTR were used, and signal intensities from the probes under
different conditions were compared to see if a 3\p UTR is longer or shorter
under certain conditions \cite{sandberg_proliferating_2008,
ji_progressive_2009}. The lengthening or shortening of a 3\p UTR would
correspond to the choice of upstream or downstream polyadenylation sites. Since
microarrays could be used to directly measure 3\p UTR length and expression
levels under different experimental condition, a broader range of questions
than before could now be tested. 

Results obtained with microarray and EST data include different patterns of
polyadenylation in different human tissues \cite{zhang_biased_2005}, and
wide-spread shortening of 3\p UTR length during immune cell activation
\cite{sandberg_proliferating_2008}. A combination of EST, microarray, and SAGE
data showed a progressive lengthening of mouse 3\p UTRs during embryonic
development \cite{ji_progressive_2009}.

Microarray studies, although capable of providing a wealth of insight, are
fundamentally limited in scope when the purpose is to study alternative
polyadenylation. This is because no direct evidence of polyadenylation sites
can be obtained with microarrays, since the information is not in
sequence-format. As well, one can only use microarrays to investigate those
annotated genes which are already suspected or known to be subject to
alternative polyadenylation. Further, microarray output is only in terms of
relative differences of gene expression. This means that it is not possible to
compare 3\p UTR lengths across different genes.

With the advance of second generation sequencing technologies in the form of
RNA sequencing (RNA-seq) in the late 2000s, many of the limitations of both EST
and microarray data seem to have been resolved. RNA-seq combines the best of
EST and microarray data when studying alternative polyadenylation. Firstly,
like ESTs, the RNA-seq data is in sequence format, allowing the direct
detection of poly(A) tails and thereby the site of cleavage and
polyadenylation. Secondly, like EST data, RNA-seq data is quantitative,
allowing the direct comparison of expression levels of 3\p UTRs across the
genome. And thirdly, like microarrays, RNA-seq can easily be performed on RNA
samples from different conditions in the same experiment, allowing direct
hypothesis testing which cannot be done when using only information from
databases. 

RNA-seq was rapidly used to study the polyadenylation landscape for cell lines
and tissues. These experiments confirmed what had been discovered earlier by
single-mRNA studies and EST analysis; that AAUAAA is the canonical
polyadenylation signal, that single genes can be represented with multiple
sites of polyadenylation, and that there is frequent polyadenylation of
intronic sequences. The new discoveries included many novel polyadenylation
sites scattered across the genome \cite{ozsolak_comprehensive_2010,
derti_quantitative_2012}. It was also found that intronic and intergenic
polyadenylation sites are in humans associated with a novel TTTTTTTTT motif
which does not occur at the normal polyadenylation sites in 3\p UTRs
\cite{ozsolak_comprehensive_2010}. Further, genome wide annotation of
polyadenylation sites was obtained in large scale for the first time in
\textit{C. elegans} and \textit{A. thaliana} \cite{mangone_landscape_2010,
wu_genome-wide_2011}. The last examples also show how RNA-seq can be used to
improve genome annotation.

As mentioned, microarray studies revealed unidirectional changes of 3\p UTR
length during cell development stages \cite{sandberg_proliferating_2008,
ji_progressive_2009}, hinting that 3\p UTR lengths can be regulated at a global
level. A study that did not use high-throughput data identified shortening of
the 3\p UTR of many transcripts in a cancer cell line, proposing that this was
a general characteristic in cancer cells \cite{mayr_widespread_2009-2}. As a
follow up to this study, Fu et al. compared the relative change in 3\p UTR
length between two cancer cell lines and a non-cancer control cell line
\cite{fu_differential_2011}. They did not find a consistent pattern of
shortening of 3\p UTRs in the cancer cell lines. Instead, the 3\p UTR lengths
of one of the cancer cells was shorter and 3\p UTR lengths of the other was
longer than the control. This suggests that there is no clear-cut genome-wide
trend of short 3\p UTRs in cancer cells, contrary to what had previously been
concluded.

An unexpected finding with RNA-seq has been the detection of poly(A) tails for
histone mRNA in both human, mice, and \textit{C. elegans}
\cite{mangone_landscape_2010-1, shepard_complex_2011}.  Histone mRNA were
previously thought to be the only mRNA in metazoans without a poly(A) tail
\cite{marzluff_metabolism_2008}, even though several of the histone genes had
been found to contain the AATAAA polyadenylation signal at the 3\p end
\cite{keall_histone_2007}. A possible explanation for the discovery of histone
mRNA with poly(A) tails is that the histone transcripts are first cleaved and
polyadenylated, and subsequently processed to lose their poly(A) tail
\cite{mangone_landscape_2010-1} (in the supplementary materials
\cite{mangone_landscape_2010-1}). Another possibility is that if some histone
mRNA are polyadenylated, they avoid degradation at the end of the S-phase
(histone mRNA is usually degraded during the S-phase of the cell cycle), and
may thus be translated throughout the cell cycle \cite{shepard_complex_2011}.

The most recent and thorough study of genome-wide polyadenylation was done
using a novel sample-preparation protocol by Derti et al. They used RNA-seq to
find polyadenylation sites in five mammal species, including human, in 24
tissues \cite{derti_quantitative_2012}. They found over 400.000 polyadenylation
sites in the human tissues, compared to 150.000 found previously. However, how
many of these sites correspond to normal mRNA 3\p stable poly(A) tail or
degradation-related transient poly(A) tails was not investigated. One reason
why they have found so many sites compared to previous studies could be the
increased resolution in this study: most novel polyadenylation sites were found
in lowly expressed transcripts, which may not previously have been detected.
Derti et al. also found that although many poly(A) sites were tissue specific,
70 \% of genes showed the same usage of alternative polyadenylation across all
tissues.
