% This is just for letting me cite stuff.
%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}

% What to include? Background on transcription in eukaryotes, polyadenylation,
% and rna-seq.

\subsection{Terminology}
\begin{itemize}
	\item Read -- sequence fragment from a sequencing machine
	\item RNA-seq -- high throughput RNA sequencing
	\item PAS -- Polyadenylation signal, for example AAUAAA
	\item Poly(A) cluster -- site in the genome where polyadenylated reads
		cluster together
\end{itemize}

\subsection{Eukaryotic transcription termination and polyadenylation}

\subsubsection{Eukaryotic transcription termination}
In eukaryotes transcription happens inside the nucleus, out of reach of
the ribosomes in the cytoplasm. Therefore, translation in eukaryotes does not
occur co-transcriptionally like it does in bacteria.

\subsubsection{Cleavage and polyadenylation marks the 3' end of messenger RNAs}
Polyadenylation is the addition of a poly-adenine tail to an RNA molecule by a
poly(A) polymerase and is pervasive in the kingdom of life. In bacteria,
poladenylation is used for targeting an mRNA for degradation. In eukaryotes,
polyadenylation occurs primarily in the nucleus, co-transcriptionally, and
facilitates transport to the cytoplasm and actually confers protection from
degradation. Further, polyadenylation is directly linked to transcription
termination. Unlike in bacteria, in eukaryotes the site of transcription
termination is not the same as the site of the 3' end of the mRNA. Rather,
eukaryotes must actively cleave the transcript at a designated site while the
RNA polymerase continues transcribing downstream. The cleaved 3' end is then
immediately polyadenylated by the poly(A) polymerase to obtain a poly(A) tail.
In yeast this poly(A) tail contains about 80 adenines, while the number is
about 250 in mammals. mRNAs without a poly(A) tail are not transported to the
cytoplasm but are instead rapidly degraded in the nucleus as part of the
cellular garbage collection system.

The cleave site where polyadenylation occurs is generally found in the 3'
untranslated region (3'UTR) of the mRNA. The 3' UTR of eukaryotic mRNA is often
longer than both the 5'UTR and the coding sequence (CDS). One reason for the
length of the 3'UTR is that it contains many regulatory sequences and
structures that participate in regulation of mRNA stability, translatability,
and transport.

Not only does the 3'UTR hold many regulatory regions, but it is also itself
regulated in terms of its length. A given mRNA typically contains multiple
sites where cleavage and polyadenylation is possible. Most of these sites are
in the 3'UTR, but there are also examples of such sites in introns. By varying
the length of a given 3'UTR, regulatory regions can either be included or
excluded. Examples of regulation by 3UTR length is (XXX insert a bunch of
examples here, you've saved them).

\subsubsection{Signals that direct cleavage and polyadenylation}
Early experiments of gene sequencing identified a consensus AAUAAA sequence
between 10 and 30 nucleotides upstream the cleavage and polyadenylation sites.
Later, other such polyadenylation signals (PAS) were found. Recent
investigations have found that 70\% of polyadenylation sites contain the
canonical AAUAAA sequence, 20\% contain the alternative AUUAAA, while 5\%
contain mostly single nucleotide variations of these. Some polyadenylation
sites are not associated with a PAS, and examples, which seem to have been
evolved for special regulatory purposes, have been found for PAS-independent
polyadenylation (CIT).

In addition to the PAS, there are auxiliary upstream and downstream elements
that aid in cleavage and polyadenylation. The most prominent of these is the GU
rich downstream sequence (ACRONYM). Together, these sites recruit the cleavage
and polyadenylation complexes. The exact distance from the PAS to the site of
cleavage is both sequence dependent and stochastic. In this way, the exact
cleavage site can vary up to 40 nucleotides for each mRNA.

\subsection{High throughput RNA sequencing}
\subsubsection{Large scale studies of regulation of polyadenylation}
Several genome-wide studies of polyadenylation patterns have been made. From
the early 90s onward, more and more human expressed sequence tags (ESTs) became
available. ESTs are parts of mRNA sequenced by Sanger sequence from cDNA. This
facilitated large scale analysis of 3'UTRs and polyadenylation sites. However,
these studies were limited in that the ESTs come from different cell lines
grown under different experimental conditions. With EST data it was not
possibler to study the genome wide changes in selection of poly(A) sites for a
single experiment.

With the advance of second generation sequencing technologies, this picture has
changed. These technologies were fast used to study the polyadenylation
landscape for a single cell culture. These experiments confirmed what had been
discovered earlier by single-mRNA studies and EST analysis; that AAUAAA is the
canonical polyadenylation signal and that single genes can be represented
with multiple sites of polyadenylation. However, they also brought new insight
that could only be obtained with the genome wide scale. For example, trends
were found for overall shortening of 3'UTR lengths during certain stages of
cell proliferation and for cancer cells. Most of these conclusions were reached
with early studies and must be repeated to have their validity ascertained. For
example, a recent study did not find evidence for 3'UTR shortening in cancer
cells, indicating that the early studies could have been a bit too optimistic
in their conclusions.

\subsubsection{Preparing a cDNA library for RNA-sequencing}
The work presented in this thesis and most other high-throughput studies of
polyadenylation relies on data made with Illumina sequencing technologies.

To obtain good sequencing results, fragmented DNA in large concentrations is
required. To sequence RNA, therefore, it is necessary to both amplify the
cellular RNA extract and to convert the RNA to cDNA by reverse transcription.
At each step going from the raw RNA extract to the sequencing-ready cDNA
library, biases can be introduced depending on the choices of methods and the
order in which they are performed \cite{wang_rna-seq:_2009}. From the point of
view of polyadenylation, it is necessary that the poly(A) tail is preserved
during cDNA library preparation. This requires the use of poly(T) primers both for
making the cDNA library and for PCR amplification.

After cDNA preparation and  several rounds of PCR amplification, most poly(A)
tails will be reduced in length from the full 250 nt. This is because the
poly(T) primers will bind randomly in the poly(A) tail, something which will
gradually reduce the length of the tail.

\subsubsection{Sequencing errors and biases}
There are several errors and biases that creep into sequencing that must be
taken into account during the downstream computational analysis of the output.

An example of an error is that sequencing performance degrades during
sequencing increases over single-nucleotide stretches. This implies that when
sequencing the poly(A) sequence, other nucleotides than just A will be
reported. This means that when looking for poly(A) stretches in the output as
signatures of poly(A) sites, some variation from the mono-A sequence must be
expected.

An example of a bias is that the sequencing reads that cover poly(A) sites will
almost invariably be from the reverse-transcribed strand of the mRNA. This is
because sequencing, like transcription, happens in the 5'-> 3' direction, and
only the ends of the cDNA fragments are sequenced. Therefore, in practise, the
poly(A) stretches in the sequence output will, at least for Illumina sequencing
where the fragment length is more than twice the read length, result in
poly(T) output from the 5'-> 3' end. From now on, we will in general refer to
the poly(A/T) tail only as the poly(A) tail, even if the sequence in the reads
will often be poly(T).

\section{Methods - Utail}
We wanted to obtain information about sites of polyadenylation for RNA-seq
experiments so that we could rapidly analyze the polyadenylation pattern for
any RNA-seq experiment. To do that, we developed the tool \textit{Utail} (from
Untranslated poly(A) tail). If an RNA library has been prepared in a way that
ensures that the sites of polyadenylation will be included, Utail can use the
RNA-seq data from that library to locate and quantify the polyadenylation sites
and provide ready-to-use output.

\subsection{Utail pipeline overview}
The RNA-seq data is processed in a series of steps to go from raw data to
polyadenylation sites. The minimum requirements for Utail to run is mapped
RNA-seq reads. Presently, the reads must be provided either in the output
format of the GEM mapper CIT or as bed-files (see XXX for input
specifications). Although not necessary, the Utail output will be improved if a
genome annotation (in Ensemble or Gencode format) is supplied. The version of
the reference genome for the annotation must be the same as was used to map the
reads to in the first place.

Utail takes the mapped RNA-seq data through the following pipeline (each point
is elaborated on below):
\begin{enumerate}
	\item (Only if annotation was supplied) Create a list of all possible
		3'UTR regions and polyadenylation sites from the genome annotation
	\item Filter all reads and retain only the unmappable reads
	\item Keep only the unmappable reads with leading poly(T) or trailing
		poly(A) stretches
	\item Remove the poly(T) or poly(A) stretch from the read and remap the
		remaining part to the genome -- keep the uniquely mapping ones
	\item Determine the site of polyadenylation depending on poly(A/T) and which
		strand the read originated from
	\item Cluster together all reads that map within 40 nt of each other
	\item For each cluster, get the genomic sequence up to 60 nt downstream
		from the polyadenylation site. Note any PAS and discard cluster if a
		genomic poly(A) region is found at the site of the cleaved poly(A)
		stretch.
	\item For each poly(A) cluster, write to the output
\end{enumerate}
See Figure X for a graphical representation of the pipeline.

\textbf{1.)} By providing an annotation of the transcripts from the genome
re-map the read back to the genome. If the read maps uniquely (with up to 2
mismatches), consider this the 3' end of the mapping a putative poly(A) site.

\textbf{5.)} Even though the sequencing protocol is not strand-specific, we can
use the information from the poly(A) versus poly(T) stretch to find out if the
read has been reverse transcribed or not. For example, if the read maps to the
+ strand with a poly(A) tail, the read originated from the + strand. If the
read maps to the + strand with a poly(T) tail, the read originated from the -
strand. This allows the strand-specific determination of poly(A) sites.

\textbf{6.)} Since the exact site of cleavage and polyadenylation is stochastic,
we must cluster the different poly(A) sites. The average of this clustering is
reported as the cleavage and polyadenylation site for all reads in the cluster.

\textbf{7.)} The sequence surrounding the putative poly(A) site is searched for
one of the several reported PAS sequences as well as for a genomic poly(A)
stretch that could explain the poly(A) stretch thought to be a result of
polyadenylation. in
question, the set of already-annotated polyadenylation sites can be
constructed. If this step is included, it will be included in the output
whether a discovered polyadenylation site has already been annotated or not.

\textbf{2.)} With default settings, short-read mappers will not be able to map
to the genome a read-fragment containing the non-genomic poly(A)
stretch. Therefore, we are only interested in those reads which were initially
unmapped. The output format of mappers usually contains a flag which says if
the read was mapped or not.

\textbf{3.)} Among the unmapped reads, we are only interested in the ones which
contain a stretch of poly(A) at the 3' end or poly(T) at the 5' end. A minimum
of 5 poly(A/T) or 5 poly(A/T) in 7 was used.

\textbf{4.)} Once the poly(A) tail has been removed, we use the GEM mapper to
re-map the read back to the genome. If the read maps uniquely (with up to 2
mismatches), consider this the 3' end of the mapping a putative poly(A) site.

\textbf{5.)} Even though the sequencing protocol is not strand-specific, we can
use the information from the poly(A) versus poly(T) stretch to find out if the
read has been reverse transcribed or not. For example, if the read maps to the
+ strand with a poly(A) tail, the read originated from the + strand. If the
read maps to the + strand with a poly(T) tail, the read originated from the -
strand. This allows the strand-specific determination of poly(A) sites.

\textbf{6.)} Since the exact site of cleavage and polyadenylation is stochastic,
we must cluster the different poly(A) sites. The average of this clustering is
reported as the cleavage and polyadenylation site for all reads in the cluster.

\textbf{7.)} The sequence surrounding the putative poly(A) site is searched for
one of the several reported PAS sequences as well as for a genomic poly(A)
stretch that could explain the poly(A) stretch thought to be a result of
polyadenylation.

\textbf{8.)} The output for each cluster contains information about the
chromosome, strand, number of reads in the cluster, if the site has been
annotated, and if there is a PAS withing 40 nucleotides and the distance to
this PAS.

\subsection{Requirements and repository}
Utail can be obtained from its Git repository git@github.com:jorgsk/Utail.git.
In runs in a Linux/Unix environment and has the following requirements:
\begin{itemize}
	\item Python 2.6 ++
	\item BEDTools, a set of programs for manipulating .bed files
		\cite{quinlan_bedtools:_2010}
\end{itemize}
In addition, the following dependencies come bundled with Utail:
\begin{itemize}
	\item pyFasta, a Python module by Brent Pedersen for rapid extraction of
		genomic sequences from a reference genome
	\item bedGraphToBigWig, a program that converts bedGraph files to bigWig
		files, obtained from the ucsc.edu website.
\end{itemize}
Before running the program, modify the settings in he $UTR_SETTINGS$ file to your
preferences. Here you provide links to the RNA-seq data and the annotation and
set various settings. Run using ``python utail.py''; after completion the
output will be in the automatically created ``output'' folder.

\subsection{The dataset}
The datasets used in this study was generated by the ENCODE
(\textbf{Enc}yclopedia \textbf{O}f \textbf{D}NA \textbf{E}lements) consortium
and are available from http://hgdownload-test.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeCshlLongRnaSeq/

Table \ref{tab:Datasets} shows the cell lines and compartments used. In total,
data from 12 human cell lines were used. 23 datasets were from whole cell
extracts, 11 were from cytoplasmic extracts, and 12 were from nuclear extracts.
For each cell line and compartment, datasets are available both for the
poly(A)+ and poly(A)- fraction of the RNA pool). This brings the total to 92
RNA-seq datasets. Each dataset has been generated with Illumina paired-ended
sequencing with a read-length of 75 basepairs.
\begin{table}
	\centering
	\begin{tabular}{cccc}
	  Cell line & Whole Cell & Cytoplasm & Nucleus \\
	  \midrule
	  GM12878 & 2 & 2 & 2 \\
	  K562 & 2 & 2 & 2 \\
	  HeLa-S3 & 2 & 2 & 2 \\
	  HUVEC & 2 & 2 & 2 \\
	  HEPG2 & 2 & 2 & 2 \\
	  H1Hesc & 1 & 1 & 1 \\
	  Nhek & 2 & 0 & 1 \\
	  MCF7 & 2 & 0 & 0 \\
	  AG04450 & 2 & 0 & 0 \\
	  HSMM & 2 & 0 & 0 \\
	  NHLF & 2 & 0 & 0 \\
	  A549 & 2 & 0 & 0 \\
	\end{tabular}
	\caption{Number of replicates of the datasets from the ENCODE consortium}
	\label{tab:Datasets}
\end{table}

The RNA library for this protocal was not especially targeted toward the 3' end
of the RNAs, but since poly(T) primers have been used for amplification and
cDNA creation, it is possible to find polyadenylated RNA in these data. What
the data lacks in specificity for the poly(A) sites, it makes up for it in
quantity, with more than 90 datasets from 12 cell line.

\subsection{The short RNA mapper}
The short read mapper used in this work is the GEM mapper CIT webpage. The GEM
mapper is developed in the group of Roderic Guigo and has been used regularly
in the group the last years, although it has not been published yet.

\subsection{Merging and screening of poly(A) clusters}
In the results presented here, the poly(A) clusters for all datasets were
merged. We accepted as a poly(A) site a cluster with support from 3 or more
reads.

\section{Results}
We ran Utail on RNA-seq data that was part of the ENCODE consortium. Parts of
the analysis was published in XXX. Here we outline our contribution
to the paper, including the parts of the analysis which was not included in the
final paper.

There is good material in the reports.

You are still not sure: how close to the paper should you present your results?
You should probably ask Martin about this tomorrow. Now, go to the introduction
for the other stuff.

\subsection{k}


\end{document}


