% This is just for letting me cite stuff.
%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
\subsubsection{Overview of transcription in eukaryotes}
In eukaryotes there are several RNA polymerases, each one responsible for
transcribing different classes of genes. The most abundant polymerase is 
RNAP II which transcribes mostly protein coding genes. With 12 subunits in
mammals, RNAP II is larger than its bacterial counterpart, but the core
subunits are structurally and functionally conserved compared to bacterial RNAP
\cite{ebright_rna_2000}.

As in bacteria, transcription in eukaryotes begins when RNAP II is recruited by
transcription factors to promoter sites, although this recruitment is more
diversified than in bacteria as no equivalent to the sigma factor system
exists in eukaryotes. As well, transcription initiation involves the same steps
of open bubble formating and abortive cycling, although there are more
assisting transcription factors are involved in this process compared to
bacteria \cite{wade_transition_2008}. Transcription elongation by RNAP II also
involves pausing and backtracking, but pausing on eukaryotic DNA is more
complicated due to wrapping of DNA in nucleosomes and chromatin
\cite{sims_elongation_2004}.

Transcription termination on the other hand is markedly different between
eukaryotes and bacteria. In bacteria, the position where RNAP stops
transcribing also marks the genomic position of the 3\p end of the mRNA. In
eukaryotes however, transcription termination does not co-occur with mRNA
release and 3\p end definition. Instead, mRNA is released by a process of
cleavage and polyadenylation several hundred nucleotides before RNAP terminates
transcription.

\subsubsection{Cleavage and polyadenylation marks the 3' end of mRNA
and promotes transcription termination}
There are usually several sites where cleavage and polyadenylation can occur in
a gene, though these sites are predominantly found in the 3\p UTR. The cleavage
and polyadenylation sites are identified by the polyadenylation signal (PAS),
which is a highly conserved AATAAA signal sequence or a close variant like
ATTAAA, in addition to an enhancing U or GU rich sequence further downstream
the PAS. The CPSF (cleavage/polyadenylation specificity factor) protein binds
to the PAS along with several other factors to form a poly(A) binding complex.
The complex then cleaves mRNA in a stochastic manner 10-30 nucleotides
downstream the PAS (see \cite{proudfoot_ending_2011} for a comprehensive and
historical review of 3\p cleavage and polyadenylation). As soon as mRNA has
been cleaved, a poly(A)-polymerase will add a polyA tail (length 250 in
mammals) to the 3\p end and the tail will be covered by poly(A) binding protein
(PAB). The PAB protects the poly(A) tail from degradation and assists in
transporting the mRNA to the cytoplasm \cite{mangus_poly-binding_2003}.

As cleavage and polyadenylation occur, RNAP II is still transcribing the
downstream DNA. It is important that RNAP II terminates transcription,
otherwise it can collide with the polymerases that transcribe downstream genes.
One characterized method of transcription termination is by a 5\p to 3\p
exonuclease called XRN2. In what is called the ``torpedo model'', this
exonuclease degrades the still-transcribed RNA from the 5\p end until it
reaches RNAP II and causes transcription termination, although the exact
details of how transcription termination happens is not clear
\cite{kuehner_unravelling_2011}.

\subsubsection{mRNA processing is necessary step before translation}
In contrast to bacterial transcripts, eukaryotic mRNA in mammalian cells needs
extensive processing before it is capable of being translated by ribosomes.
Therefore, mRNA in eukaryotes is called pre-mRNA until it has gone through the
necessary processing steps. There are essentially three main events that make
up pre-mRNA processing: 5\p cap addition, splicing, and 3\p cleavage and
polyadenylation.

The 5\p cap is a guanine nucleotide variant that is added to the 5\p nucleotide
of a transcript shortly after it emerges from the RNA exit channel of RNAP II.
The 5\p cap presumably protects against 5\p exonucleases and is necessary for
the proper export of the mRNA from the nucleus.

Splicing is the removal of introns, the non-coding parts of pre-mRNA that make
up most of gene sequences in mammals. By splicing out the introns and joining
the surrounding exons, only the coding regions (as well as the 3\p and 5\p
untranslated regions) of pre-mRNA make up the final mRNA.

Each of the steps of pre-mRNA processing are interconnected. The 5\p cap
increases the efficiency of the excision of the 5\p proximal intron and the
efficiency of polyadenylation; and polyadenylation again increases the
efficiency of the excision of the 3\p terminal intron
\cite{proudfoot_integrating_2002}. Because of this interconnectedness, some
proteins have shared roles (they cross-talk) across the three processing
pathways. As an example of this, it was recently found that the U1
ribonucleoprotein, which was previously known for its role in splicing,
prevents premature cleavage and polyadenylation at cryptic polyadenylation
signals \cite{kaida_u1_2010}.

If any of the processing steps are inefficiently executed or not executed at
all, the pre-mRNA transcript will be targeted for degradation either in the
nucleus itself or in the cytoplasm after transport \cite{doma_rna_2007}. This
reduces the risk that errors during transcription and pre-mRNA processing will
result in nonfunctional or possibly harmful protein products.

\subsubsection{Usage of alternative polyadenylation sites}
% 1) For making different 3 UTR
Each 3\p UTR region of a gene often has several alternative sites where
cleavage and polyadenylation may happen \cite{tian_large-scale_2005}. Depending
on which of these sites are used for cleavage and polyadenylation, the length
of the 3\p UTR in the final mRNA will vary. This can affect the downstream fate
of the mRNA as the 3\p UTR region contains many regulatory elements, such as
binding sites for microRNA. MicroRNAs that bind the 3\p UTR region in the
cytoplasm generally decrease expression from the mRNA they bind to by inducing
mRNA degradation \cite{huntzinger_gene_2011}. Since microRNAs are involved in a
host of human diseases and other metabolic processes
\cite{huang_biological_2010}, the choice of where to cleave and polyadenylate a
transcript can have wide-spanning consequences.

There are also examples of cleavage and polyadenylation in sites outside the
3\p UTR. The most prominent of these are sites within introns. The use of an
intronic cleavage and polyadenylation site a site would cause any downstream
exons to be left out of the transcript. Thus, the site of cleavage and
polyadenylation can also modulate the protein coding content of the mRNA. A
well known example is from the immunoglobin protein in B cells. Depending on
whether a polyadenylation site in an intron is used or not, the membrane bound
or the secreted version of this protein is made \cite{peterson_regulated_1989}.

\subsubsection{Transient polyadenylation unrelated to mRNA processing}
In the last decade, it has become clear that there is polyadenylation of
RNA in eukaryotic cells that is unrelated to mRNA 3\p processing. First, it was
discovered that short transient polyadenylation occurred in aberrant
transcripts in the nucleus of yeast by the TRAMP complex. The transcripts
polyadenylated by TRAMP were found to be targeted for degradation by the
exosome \cite{lacava_rna_2005, wyers_cryptic_2005}. This is similar to the
situation in bacteria, where polyadenylation is part of the RNA degradation
pathway. This new type of polyadenylation however is oposite of the role of
stable polyadenylation of mRNA in the cytoplasm, where the poly(A) tail
protects against degradation. It was suggested that degradation-associated
polyadenylation by TRAMP has been conserved from the original bacterial
degradation machinery \cite{lacava_rna_2005}. Later, degradation-related
polyadenylation was found in the nucleus of mammals too, and eventually even in
the cytoplasm of human cells \cite{slomovic_polyadenylation_2006,
slomovic_addition_2010}. In summary, there is an emergent role for
degradation-related transient polyadenylation in eukaryotes that was previously
only thought to occur in bacteria.

\subsubsection{Genome-wide studies of polyadenylation}
The study of sites of polyadenylation across the genome has occurred in three
stages in the last 12 years, with each stage resting on a different type of
technology. The first wave used cDNA and EST sequence data obtained by
laborious Sanger sequencing. The second wave used microarray and SAGE
technologies, and the third wave used the RNA-seq next generation sequencing
technology. We will review key results obtained in these three stages
chronologically.

From the early 90s onward, more and more human expressed sequence tags (ESTs)
and cDNA became available. The increasing amount of sequence data facilitated
for the first time large scale analysis of 3\p UTRs and polyadenylation sites.
The polyadenylation site of a cDNA sequence is found by identifying the poly(A)
or poly(T) extremity which does not correspond to a genomic sequence, trimming
that extremity, and matching the remainder to template mRNA or genomic sequence
\cite{beaudoing_patterns_2000, tian_large-scale_2005}.

These early genome-wide studies were successful in determining i) the frequency
of occurrence of the different PAS \cite{beaudoing_patterns_2000}, ii) that
over half of human genes employ alternative polyadenylation, iii) that sites of
alternative polyadenylation are evolutionary conserved between humans and mice
\cite{tian_large-scale_2005}, and vi) that there is extensive polyadenylation
at intronic polyadenylation sites \cite{tian_widespread_2007}.

However, these studies were limited in the type of questions they could ask.
Crucially, they could not compare different experimental conditions, since the
ESTs are from different cell lines grown under different experimental
conditions. Further, the EST data is biased toward protein coding genes that
were found interesting enough to sequence individually. Thus many classes of
polyadenylated RNA, such as long noncoding RNA, were possibly missed by these
studies. Finally, no information about gene expression levels can be had from
EST data, so was is not possible to find out how alternative polyadenylation
could affect gene expression.

As the microarray technology matured and more full-length genomes became
available, microarrays were used to study 3\p UTR variation and
polyadenylation, often in combination with EST data. Now, alternative
polyadenylation could be investigated between different experimental
conditions. As well, 3UTR expression levels could be compared across these
experiments. This allowed for a broader range of hypotheses to be tested. In
practise, to test for differential 3UTR usage with microarrays, RNA probes
corresponding to the full length or extended 3UTR were used, and signal
intensities from the probes under different conditions could be compared to see
if a 3UTR is longer or shorter under certain conditions
\cite{sandberg_proliferating_2008, ji_progressive_2009}. The lengthening or
shortening of a 3\p UTR would correspond to the choice of upstream or
downstream polyadenylation sites.

Results obtained with microarray and EST data include different patterns of
polyadenylation in different human tissues \cite{zhang_biased_2005}, and
wide-spread shortening of 3\p UTR length during immune cell activation
\cite{sandberg_proliferating_2008}. A combination of EST, microarray, and SAGE
data (SAGE being similar to microarrays) showed a progressive lengthening of
mouse 3\p UTRs during embryonic development \cite{ji_progressive_2009}.

Microarrays studies, although capable of providing a wealth of insight, are
fundamentally limited in scope when the purpose is to study alternative
polyadenylation: since no direct evidence of polyadenylation events can be had
with microarrays, one can only investigate those annotated genes which are
already suspected or known to be subject to alternative polyadenylation.
Further, due to the nature of microarray data, 3\p UTR length differences are
only reported as relative differences between experiments, making it impossible
to compare 3\p UTR lengths across genes.

With the advance of second generation sequencing technologies in the late 2000s
in the form of RNA sequencing (RNA-seq), both limitations for EST and
microarray data seem to have been resolved. RNA-seq combines the best of EST
and microarray data when studying alternative polyadenylation. Firstly, like
ESTs, the RNA-seq data is in sequence format, allowing the direct detection
of poly(A) tails and thereby the site of cleavage and polyadenylation.
Secondly, like microarrays, RNA-seq can be performed on RNA samples from
different experimental conditions, allowing direct hypothesis testing. And
further, unlike both ESTs and microarray, RNA-seq data is truly quantitative,
allowing the direct comparison of expression levels of 3\p UTRs across the
genome.

RNA-seq was rapidly used to study the polyadenylation landscape for cell lines
and tissues. These experiments confirmed what had been discovered earlier by
single-mRNA studies and EST analysis; that AAUAAA is the canonical
polyadenylation signal, that single genes can be represented with multiple
sites of polyadenylation, and that there is frequent polyadenylation of
intronic sequences. Among the new discoveries were first of all many novel
polyadenylation sites scattered across the genome
\cite{ozsolak_comprehensive_2010, derti_quantitative_2012}. It was also found
that intronic and intergenic polyadenylation sites are in humans associated
with a novel TTTTTTTTT motif which does not occur at canonical polyadenylation
sites at the 3\p end of genes \cite{ozsolak_comprehensive_2010}. Further,
genome wide annotation of polyadenylation sites from sense and antisense
transcription was obtained for the first time in large scale in \textit{C.
elegans} and \textit{arabidopsis} \cite{mangone_landscape_2010,
wu_genome-wide_2011}. The last examples show how RNA-seq is directly used in
genome annotation. In a follow up to microarray-studies of change in global 3\p
UTR length in cancer, Fu et al. compared the relative change in 3\p length
between two cancer cells and an epithelial cell line
\cite{fu_differential_2011}. They did not find a consistent pattern of
widespread shortening of 3\p UTRs in the cancer cell lines, a phenomenon which
had been previously suggested using a restricted dataset of 23 specifically
selected genes \cite{mayr_widespread_2009-2}. An unexpected finding with
RNA-seq has been the detection of poly(A) tails for histone mRNAs in both human
and \textit{c.  elegans} \cite{mangone_landscape_2010-1, shepard_complex_2011}.
Histone mRNAs were previously thought to be the only mRNAs in metazoans without
a poly(A) tail \cite{marzluff_metabolism_2008}, even though several of the
histone genes had been found to contain the AATAAA polyadenylation signal at
the 3\p end \cite{keall_histone_2007}.

The most recent and thorough study of genome-wide polyadenylation was done
using a novel sample-preparation protocol by Derti et al. They used RNA-seq to
find polyadenylation sites in five mammal species, including human, in 24
tissues \cite{derti_quantitative_2012}. They found over 400.000 polyadenylation
sits in the human tissues, compared to 150.000 found previously; although how
many of these sites correspond to mRNA 3\p stable poly(A) tail or
degradation-related transient poly(A) tails was not investigated. On reason for
this very large number of sites could be that they identified most novel
polyadenylation sites in lowly expressed transcripts, which may not previously
have been detected. They also found that although many poly(A) sites were
tissue specific, 70 \% of genes showed the same usage of alternative
polyadenylation across all tissues.

\subsubsection{Preparing a cDNA library for RNA-sequencing}
The work presented in this thesis and most other high-throughput studies of
polyadenylation rely on sequencing data made with Illumina sequencing
technologies. Here we briefly mention the stages of an RNA-seq
experiment that are relevant for interpreting subsequent results.

To obtain good sequencing results, fragmented DNA in large concentrations is
required. To reach these concentration, it is generally necessary to both
amplify the cellular RNA extract and to convert the RNA to cDNA by reverse
transcription. Both RNA amplification and cDNA conversion can introduce biases
depending on the choices of methods and the order in which they are performed
\cite{wang_rna-seq:_2009}. From the point of view of studying polyadenylation,
it is necessary that the poly(A) tail is preserved during cDNA library
preparation. This requires the use of poly(T) primers both for making the cDNA
library and for PCR amplification.

After cDNA preparation and several rounds of PCR amplification, most poly(A)
tails will be reduced in length from the full 250 nt. This is because the
poly(T) primers will bind randomly in the poly(A) tail, something which will
gradually reduce the average length of the poly(A) tails.

\subsubsection{Sequencing errors and biases}
There are several errors and biases that creep into sequencing that must be
taken into account during the computational analysis of the output. Here we
discuss one error and one bias that is important for analysing Illumina short
pair ended reads.

An error that is particularly important for the study of polyadenylation is
that sequencing performance degrades over single-nucleotide stretches
\cite{minoche_evaluation_2011}. This implies that when sequencing the poly(A)
sequence, other nucleotides than just A will be reported with a relatively high
frequency.. This means that when looking for poly(A) stretches in the output as
signatures of poly(A) sites, some variation from the homopolymer sequence must
be expected.

An important bias is that most sequences with poly(A) tails will be output in
the reverse-transcribed version. That is, 5\p CCCGAAAA 3\p will be output as 5\
TTTTCGGG 3\p. This is due to the RNA fragmenting and that sequencing
(performed by a DNA polyermase) happens in the 5\p to 3\p direction and that
each RNA is present in both its original and reverse transcribed form. During
sequencing, 50 to 150 nucleotides will be sequenced from each side of the around
300 nt fragment. If the poly(A) tail is the 50 last nucleotides of the 300 nt
fragment RNA, sequencing 150 nucleotides will never reach nucleotide 250 where
the poly(A) site is. However, the reverse transcribed version contains 50 Ts at
the 5\p end, which ensures that sequencing this version of the RNA gets the
whole poly(T) tail as well as 100 bases which can be mapped to the genome.

Even though the actual output poly(A) tail is in the reverse transcribed
poly(T) format, we will often refer to both of them as a poly(A) tail.

\subsubsection{Mapping reads to the genome}
The sequence-snippets that are output from sequencing machines are called
reads, and generally come in sizes from 30 to 500 basepairs, depending on the
technology used. If a reference genome exists for the organism from which the
RNA sample was taken, these reads must be mapped to that genome. Otherwise, it
is possible to attempt to construct the organisms transcriptome \textit{de
novo} from the RNA-seq output, although this generally gives poor results.

Mapping read to the genome means trying to find where the RNA-snippet from the
sequencing machine originated from. Different methods are used for this, but
what they have in common is that they allow for mismatches between the read and
the genome to allow for sequencing errors. Extra care is taken to map reads
spanning the intron-exon junctions, since these reads must necessarily be split
in two to be mapped, possibly thousands of nucleotides apart.
