% This is just for letting me cite stuff.
%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
\subsubsection{Transcription and transcript processing in mammals}
In eukaryotes there are several RNA polymerases, each one responsible for
transcribing different classes of genes. The most abundant polymerase is 
RNAP II and transcribes mostly protein coding genes. RNAP II is larger than its
bacterial counterpart, with 12 subunits in mammals, but the core subunits are
structurally and functionally conserved compared to bacterial RNAP
\cite{ebright_rna_2000}.

As in bacteria, transcription in eukaryotes begins at promoter sites, and
transcription initiation involves the same steps of open bubble formating and
abortive cycling, although many more assisting transcription factors are
involved \cite{wade_transition_2008}. Transcription elongation by RNAP II also
involves pausing and backtracking, but pausing on eukaryotic DNA is more
complicated due to the presence of chromatin and its methylation or acetylation
states \cite{sims_elongation_2004}. Transcription termination is markedly
different between eukaryotes and prokaryotes and will be covered more
extensively below.

In contrast to bacterial transcripts, mRNA in mammalian cells needs
extensive processing before it is capable of being translated by ribosomes.
Therefore, mRNA in eukaryotes is called pre-mRNA until it has gone through the
necessary processing steps. There are essentially three main events that make
up mRNA processing: 5\p cap addition, splicing, and 3\p cleavage and
polyadenylation.

The 5\p cap is a guanine nucleotide variant that is added to the 5\p nucleotide
of a transcript as it emerges from the RNA exit channel of RNAP II. The 5\p
cap presumably protects against 5\p exonucleases and aids in the splicing of
the 5\p intron.

Splicing is the removal of parts of pre-mRNA called introns. Introns are
non-coding parts of genes that actually make up most of gene sequences in
mammals. By splicing out the introns and joining the surrounding exons, only
the protein coding parts of the gene ends up in the final mRNA.

Cleavage and polyadenylation mark the 3\p end of the pre-mRNA. At designated
sites, the nascent transcript is cleaved in two while still being transcribed
by RNAP II. The 3\p end of the pre-mRNA is then catalytically extended with a
tail of around 250 adenosione nucleotides; this is generally the last step of
mRNA processing and marks the transition from pre-mRNA to mRNA. Below, we will
review in more detail the process of cleavage and polyadenylation.

Eeach of the steps of pre-mRNA processing are interconnected. The 5\p cap
increases the efficiency of the excision of the 5\p proximal
intron and the efficiency of polyadenylation; polyadenylation again increases
the efficiency of the excision 3\p terminal intron
\cite{proudfoot_integrating_2002}. Because of this interconnectedness, some
proteins have shared roles across the three processing pathways. For example,
it was recently found that the U1 snRNP, which was previously known for its
role in splicing, prevents premature cleavage and polyadenylation at cryptic
polyadenylation signals \cite{kaida_u1_2010}.

If any of the processing steps are inefficiently executed or not executed at
all, the pre-mRNA transcript will be targeted for degradation either in the
nucleus itself or in the cytoplasm \cite{doma_rna_2007}. This reduces the risk
that errors during transcription and pre-mRNA processing will result in
unfunctional or possibly harmful protein.

\subsubsection{Cleavage and polyadenylation marks the 3' end of messenger RNAs
and promotes transcription termination}
Most protein coding genes contain a 3\p untranslated region (3\p
UTR) which may contain several sites where cleavage and polyadenylation can
occur. These sites are identified by the polyadenylation signal (PAS), which is
a highly conserved AAUAAA signal sequence or a close variant like AUUAAA. The
protein CPSF (cleavage/polyadenylation specificity factor) binds to the PAS and
with the help of other factors cleaves pre-mRNA 10-30 nucleotides downstream
the PAS. As soon as pre-mRNA has been cleaved, a poly(A)-polymerase will add
the poly(A) tail and the tail will be covered by poly(A) binding protein (PAB).

While the mRNA is now created, RNAP II is still transcribing downstream. One
characterized method of transcription termination is by a 5\p to 3\p
exonuclease called XRN2. In what is called the ``torpedo model'', this
exonuclease degrades the still-transcribed RNA until it reaches RNAP II and
causes transcription termination \cite{kuehner_unravelling_2011}.

\subsubsection{Usage of alternative polyadenylation sites}
% 1) For making different 3 UTR
Each 3\p UTR region of a gene may have several alternative sites where cleavage
and polyadenylation may happen. By chosing one site or the other, the length of
the 3\p UTR in the mRNA will be different. This can affect the
downstream fate of the mRNA as the 3\p UTR region contains many regulatory
elements, for example for transport and degradation, see
\cite{lutz_alternative_2011} for a recent review. Genome-wide changes in the
pattern of usage of polyadenylation sites has been shown during cell
proliferation \cite{sandberg_proliferating_2008} and for development
\cite{hilgers_neural-specific_2011, ji_progressive_2009}. This shows that the
selective usage of polyadenylation sites is part of several cellular programs.

There are also examples of cleavage and polyadenylation sites inside introns.
The use of such a site would leave out downstream exons, changing the protein
coding content of the mRNA. A well known example is that the membrane bound or
secreted versions of immunoglobin in B cells differ depending on whether
cleavage and polyadenylation occurs in an intron or not.

\subsubsection{High throughput RNA sequencing and genome-wide studies of
polyadenylation}

From the early 90s onward, more and more human expressed sequence tags (ESTs)
and cDNA became available. The increasing amount of sequence data facilitated
for the first time large scale analysis of 3\p UTRs and polyadenylation sites.
However, these studies were limited from the fact that the ESTs came from
different cell lines grown under different experimental conditions. Further,
the EST data was biased toward genes that were found interesting enough to
sequence individually. Such studies were nontheless successful in determining
i) the frequency of occurrance of the different PAS
\cite{beaudoing_patterns_2000}, ii) that over half of human genes employ
alternative polyadenylation, iii) that sites of alternative polyadenylation is
conserved between humans and mice \cite{tian_large-scale_2005}, and vi) that
there is extensive polyadenylation at intronic polyadenylation sites
\cite{tian_widespread_2007}.

As the microarray technology matured, it was also applied to studies of
polyadenylation.

With the advance of second generation sequencing technologies, sequence data
was no longer scarce. These technologies were fast used to study the
polyadenylation landscape for a single cells and tissues. These experiments
confirmed what had been discovered earlier by single-mRNA studies and EST
analysis; that AAUAAA is the canonical polyadenylation signal and that single
genes can be represented with multiple sites of polyadenylation. However, they
also brought new insight that could only be obtained with the genome wide
scale. Changes during development and proliferation were already mentioned, and
in addition a trend was found for overall shortening of 3'UTR lengths in for
cancer cells \cite{mayr_widespread_2009}. Most of these results were
reached with early studies and must be repeated to have their validity
ascertained. For example, a recent study did not find evidence for 3'UTR
shortening in cancer cells \cite{fu_differential_2011}, indicating that the
results from early studies should be reproduced.

\subsubsection{Preparing a cDNA library for RNA-sequencing}
The work presented in this thesis and most other high-throughput studies of
polyadenylation rely on sequencing data made with Illumina sequencing
technologies. Here we briefly mention the stages of an RNA-seq
experiment that are relevant for interpreting subsequent results.

To obtain good sequencing results, fragmented DNA in large concentrations is
required. To sequence RNA, therefore, it is necessary to both amplify the
cellular RNA extract and to convert the RNA to cDNA by reverse transcription.
Both RNA amplification and cDNA conversion can introduce biases depending on
the choices of methods and the order in which they are performed
\cite{wang_rna-seq:_2009}. From the point of view of polyadenylation, it is
necessary that the poly(A) tail is preserved during cDNA library preparation.
This requires the use of poly(T) primers both for making the cDNA library and
for PCR amplification.

After cDNA preparation and several rounds of PCR amplification, most poly(A)
tails will be reduced in length from the full 250 nt. This is because the
poly(T) primers will bind randomly in the poly(A) tail, something which will
gradually reduce the length of the tail.

\subsubsection{Sequencing errors and biases}
There are several errors and biases that creep into sequencing that must be
taken into account during the computational analysis of the output.

An example of an error is that sequencing performance degrades during
sequencing increases over single-nucleotide stretches. This implies that when
sequencing the poly(A) sequence, other nucleotides than just A will be
reported. This means that when looking for poly(A) stretches in the output as
signatures of poly(A) sites, some variation from the mono-A sequence must be
expected.

An example of a bias is that the sequencing reads that cover poly(A) sites will
almost invariably be from the reverse-transcribed strand of the mRNA. This is
because sequencing, like transcription, happens in the 5'-> 3' direction, and
only the ends of the cDNA fragments are sequenced. Therefore, in practise, the
poly(A) stretches in the sequence output will, at least for Illumina sequencing
where the fragment length is more than twice the read length, result in
poly(T) output from the 5'-> 3' end. From now on, we will in general refer to
the poly(A/T) tail only as the poly(A) tail, even if the sequence in the reads
will often be poly(T).

\end{document}


