% This is just for letting me cite stuff.
%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
\subsubsection{Transcription and transcript processing in mammals}
In eukaryotes there are several RNA polymerases, each one responsible for
transcribing different classes of genes. The most abundant polymerase is 
RNAP II and transcribes mostly protein coding genes. RNAP II is larger than its
bacterial counterpart, with 12 subunits in mammals, but the core subunits are
structurally and functionally conserved compared to bacterial RNAP
\cite{ebright_rna_2000}.

As in bacteria, transcription in eukaryotes begins at promoter sites, and
transcription initiation involves the same steps of open bubble formating and
abortive cycling, although many more assisting transcription factors are
involved \cite{wade_transition_2008}. Transcription elongation by RNAP II also
involves pausing and backtracking, but pausing on eukaryotic DNA is more
complicated due to the presence of chromatin and its methylation or acetylation
states \cite{sims_elongation_2004}. Transcription termination is markedly
different between eukaryotes and prokaryotes and will be covered more
extensively below.

In contrast to bacterial transcripts, mRNA in mammalian cells needs
extensive processing before it is capable of being translated by ribosomes.
Therefore, mRNA in eukaryotes is called pre-mRNA until it has gone through the
necessary processing steps. There are essentially three main events that make
up mRNA processing: 5\p cap addition, splicing, and 3\p cleavage and
polyadenylation.

The 5\p cap is a guanine nucleotide variant that is added to the 5\p nucleotide
of a transcript as it emerges from the RNA exit channel of RNAP II. The 5\p
cap presumably protects against 5\p exonucleases and aids in the splicing of
the 5\p intron.

Splicing is the removal of parts of pre-mRNA called introns. Introns are
non-coding parts of genes that actually make up most of gene sequences in
mammals. By splicing out the introns and joining the surrounding exons, only
the protein coding parts of the gene ends up in the final mRNA.

Cleavage and polyadenylation mark the 3\p end of the pre-mRNA. At designated
sites, the nascent transcript is cleaved in two while still being transcribed
by RNAP II. The 3\p end of the pre-mRNA is then catalytically extended with a
tail of around 250 adenosione nucleotides; this is generally the last step of
mRNA processing and marks the transition from pre-mRNA to mRNA. Below, we will
review in more detail the process of cleavage and polyadenylation.

Eeach of the steps of pre-mRNA processing are interconnected. The 5\p cap
increases the efficiency of the excision of the 5\p proximal
intron and the efficiency of polyadenylation; polyadenylation again increases
the efficiency of the excision 3\p terminal intron
\cite{proudfoot_integrating_2002}. Because of this interconnectedness, some
proteins have shared roles across the three processing pathways. For example,
it was recently found that the U1 snRNP, which was previously known for its
role in splicing, prevents premature cleavage and polyadenylation at cryptic
polyadenylation signals \cite{kaida_u1_2010}.

If any of the processing steps are inefficiently executed or not executed at
all, the pre-mRNA transcript will be targeted for degradation either in the
nucleus itself or in the cytoplasm \cite{doma_rna_2007}. This reduces the risk
that errors during transcription and pre-mRNA processing will result in
unfunctional or possibly harmful protein.

\subsubsection{Cleavage and polyadenylation marks the 3' end of messenger RNAs
and promotes transcription termination}
Most protein coding genes contain a 3\p untranslated region (3\p
UTR) which may contain several sites where cleavage and polyadenylation can
occur. These sites are identified by the polyadenylation signal (PAS), which is
a highly conserved AAUAAA signal sequence or a close variant like AUUAAA. The
protein CPSF (cleavage/polyadenylation specificity factor) binds to the PAS and
with the help of other factors cleaves pre-mRNA 10-30 nucleotides downstream
the PAS. As soon as pre-mRNA has been cleaved, a poly(A)-polymerase will add
the poly(A) tail and the tail will be covered by poly(A) binding protein (PAB).

While the mRNA is now created, RNAP II is still transcribing downstream. One
characterized method of transcription termination is by a 5\p to 3\p
exonuclease called XRN2. In what is called the ``torpedo model'', this
exonuclease degrades the still-transcribed RNA until it reaches RNAP II and
causes transcription termination \cite{kuehner_unravelling_2011}.

\subsubsection{Usage of alternative polyadenylation sites}
% 1) For making different 3 UTR
Each 3\p UTR region of a gene may have several alternative sites where cleavage
and polyadenylation may happen. By chosing one site or the other, the length of
the 3\p UTR in the mRNA will be different. This can affect the
downstream fate of the mRNA as the 3\p UTR region contains many regulatory
elements, for example for transport and degradation, see
\cite{lutz_alternative_2011} for a recent review. Genome-wide changes in the
pattern of usage of polyadenylation sites has been shown during cell
proliferation \cite{sandberg_proliferating_2008} and for development
\cite{hilgers_neural-specific_2011, ji_progressive_2009}. This shows that the
selective usage of polyadenylation sites is part of several cellular programs.

There are also examples of cleavage and polyadenylation sites inside introns.
The use of such a site would leave out downstream exons, changing the protein
coding content of the mRNA. A well known example is that the membrane bound or
secreted versions of immunoglobin in B cells differ depending on whether
cleavage and polyadenylation occurs in an intron or not.

\subsubsection{Genome-wide studies of polyadenylation}
The study of sites of polyadenylation across the genome has occurred in three
stages in the last 12 years, each stage resting on a different type of
technology. The first wave used cDNA and EST sequence data obtained by
laborious Sanger sequencing. The second wave used microarray and SAGE
technologies, and the third wave used the RNA-seq next generation sequencing
technology. We will review key results obtained in these three stages
chronologically.

From the early 90s onward, more and more human expressed sequence tags (ESTs)
and cDNA became available. The increasing amount of sequence data facilitated
for the first time large scale analysis of 3\p UTRs and polyadenylation sites.
The polyadenylation site of a cDNA sequence is found by identifying the poly(A)
or poly(T) extremity which does not correspond to a genomic sequence, trimming
that extremity, and matching the remainder to template mRNA or genomic sequence
\cite{beaudoing_patterns_2000, tian_large-scale_2005}.

These early genome-wide studies were successful in determining i) the frequency
of occurrance of the different PAS \cite{beaudoing_patterns_2000}, ii) that
over half of human genes employ alternative polyadenylation, iii) that sites of
alternative polyadenylation is conserved between humans and mice
\cite{tian_large-scale_2005}, and vi) that there is extensive polyadenylation
at intronic polyadenylation sites \cite{tian_widespread_2007}.

However, these studies were limited in the type of questions they could ask.
Crucially, they could not compare different experimental conditions, since the
ESTs are from different cell lines grown under different experimental
conditions. Further, the EST data is biased toward protein coding genes that
were found interesting enough to sequence individually. Thus many classes of
polyadenylated RNA, such as long noncoding RNA, were possbily missed by these
studies. Finally, no information about gene expression levels can be had from
EST data.

As the microarray technology matured and more full-length genomes became
available, microarrays were used to study 3\p UTR variation and
polyadenylation, often in combination with EST data. Now, alternative
polyadenylation could be investigated between different experimental
conditions. As well, 3UTR expression levels could be compared across these
experiments. This allowd for a broader range of hypotheses to be tested. In
practise, to test for differential 3UTR usage with microarrays, RNA probes
corresponding to the full length or extended 3UTR were used, and signal
intensities from the probes under different conditions could be compared to see
if a 3UTR is longer or shorter under certain conditions
\cite{sandberg_proliferating_2008, ji_progressive_2009}. The lengthening or
shortening of a 3\p UTR would correspond to the choice of upstream or
downstream polyadenylation sites.

Results obtained with microarray and EST data include different patterns of
polyadenylation in different human tissues \cite{zhang_biased_2005}, and
wide-spread shortening of 3\p UTR length during immunce cell activeation
\cite{sandberg_proliferating_2008}. A combination of EST, microarray, and SAGE
data (SAGE being similar to microarrays) showed a progressive lengthening of
mouse 3\p UTRs during embryonic development \cite{ji_progressive_2009}.

Microarrays studies, although providing a wealth of insight, are fundamentally
limited in scope for the study of alternative polyadenylation since one can
only investigate those genes already suspected to have variation in usage of
polyadenylation sites. Further, due to the nature of microarray data, the 3UTR
length differences are only reported as relative differences between
experiments, making it impossible to compare 3UTR lengths across genes.

With the advance of second generation sequencing technologies in the late 2000s
in the form of RNA sequencing (RNA-seq), both limitations for EST and
microarray data seem to have been resolved. RNA-seq combines the best of EST
and microarray data when studying alternative polyadenylation. Firstly, like
ESTs, the RNA-seq data is in sequence format, allowing the direct detection
of poly(A) tails and thereby the site of cleavage and polyadenylation.
Secondly, like microarrays, RNA-seq can be performed on RNA samples from
different experimental conditions, allowing direct hypothesis testing. And
further, unlike both ESTs and microarray, RNA-seq data is quantitative,
allowing the direct comparison of expression levels of 3\p UTRs across the
genome.

These technologies were rapidly used to study the polyadenylation landscape for
a single cells and tissues. The new experiments confirmed what had been
discovered earlier by single-mRNA studies and EST analysis; that AAUAAA is the
canonical polyadenylation signal, that single genes can be represented with
multiple sites of polyadenylation, and that there is frequent polyadenylation
of intronic sequences. New information were many novel polyadenylation sites
and that intronic and inter-genic polyadenylation sites are in humans
associated with a novel TTTTTTTTT motif which does not occur at canonical
polyadenylation sites at the 3\p end of genes
\cite{ozsolak_comprehensive_2010}. Further, genome wide annotation of
polyadenylation sites and antisense transcription was obtained for \textit{C.
elegans} and \textit{arabidopsis} \cite{mangone_landscape_2010,
wu_genome-wide_2011}, examples of RNA-seq aiding in the annotation of genomes
which have had less attention than the human one. Fu et al. compared the
relative change in 3\p length between two cancer cells and an epithelial cell
line \cite{fu_differential_2011}. They did not find a consistent pattern of
widespread shortening of 3\p UTRs in the cancer cell lines, a phenomenon which
had been previously suggested using a restricted dataset of 23 specifically
selected genes \cite{mayr_widespread_2009-2}. The most recent number for the
number of polyadenylation sites in human is over 400.000, and while many of
them are tissue specific \cite{derti_quantitative_2012}, 70
\% of genes showed the same usage of alternative polyadenylation across 24
tissues. The same study found that most of the novel polyadenylation events are
from lowly expressed transcripts, giving a reason for why they have not been
detected until deep transcriptome sequencing became available. 

\subsubsection{Preparing a cDNA library for RNA-sequencing}
The work presented in this thesis and most other high-throughput studies of
polyadenylation rely on sequencing data made with Illumina sequencing
technologies. Here we briefly mention the stages of an RNA-seq
experiment that are relevant for interpreting subsequent results.

To obtain good sequencing results, fragmented DNA in large concentrations is
required. To reach these concentration, it is generally necessary to both
amplify the cellular RNA extract and to convert the RNA to cDNA by reverse
transcription, although some cDNA-free methods have been published
\cite{ozsolak_comprehensive_2010}. Both RNA amplification and cDNA conversion
can introduce biases depending on the choices of methods and the order in which
they are performed \cite{wang_rna-seq:_2009}. From the point of view of
studying polyadenylation, it is necessary that the poly(A) tail is preserved
during cDNA library preparation. This requires the use of poly(T) primers both
for making the cDNA library and for PCR amplification.

After cDNA preparation and several rounds of PCR amplification, most poly(A)
tails will be reduced in length from the full 250 nt. This is because the
poly(T) primers will bind randomly in the poly(A) tail, something which will
gradually reduce the average length of the poly(A) tails.

\subsubsection{Sequencing errors and biases}
There are several errors and biases that creep into sequencing that must be
taken into account during the computational analysis of the output. Here we
discuss one error and one bias that is important for analysing Illumina short
pair ended reads.

An error that is particularly important for the study of polyadenylation is
that sequencing performance degrades over single-nucleotide stretches
\cite{minoche_evaluation_2011}. This implies that when sequencing the poly(A)
sequence, other nucleotides than just A will be reported with a relatively high
frequency.. This means that when looking for poly(A) stretches in the output as
signatures of poly(A) sites, some variation from the homopolymer sequence must
be expected.

An important bias is that most sequences with poly(A) tails will be output in
the reverse-transcribed version. That is, 5\p CCCGAAAA 3\p will be output as 5\
TTTTCGGG 3\p. This is due to the RNA fragmenting and that sequencing
(performed by a DNA polyermase) happens in the 5\p to 3\p direction and that
each RNA is present in both its original and reverse transcribed form. During
sequencing, 50 to 150 nucleotides will be sequenced from each side of the around
300 nt fragment. If the poly(A) tail is the 50 last nucleotides of the 300 nt
fragment RNA, sequencing 150 nucleotides will never reach nucleotide 250 where
the poly(A) site is. However, the reverse transcribed version contains 50 Ts at
the 5\p end, which ensures that sequencing this version of the RNA gets the
whole poly(T) tail as well as 100 bases which can be mapped to the genome.

Even though the actual output poly(A) tail is in the reverse transcribed
poly(T) format, we will often refer to both of them as a poly(A) tail.

\end{document}


