%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
Chapters X and Y deal with how gene expression in bacteria is regulated by RNA
secondary structures in the ribosome binding site (RBS), mRNA stability, and
codon usage bias. Here, we review each of those topcis. We begin with a brief
account of translation initiation and then talk about how this process is
regulated by RNA secondary structures, and how this again affects RNA
stability. We then proceed to translation elongation and how this process is
affected by condon usage bias.

\subsubsection{Brief overview of translation initiation}

Here you briefly mention these things; then you go into detail.

Since bacteria do not have a nucleus the ribosome has access to the transcript
immediately after its 5' end leaves the RNA exit channel. It has been shown
that the initially translating ribosome follows closely and even pushes the
transcribing polymerase, causing them to have the same speed
\cite{proshkin_cooperation_2010}. This ribosome pushing can prevent RNAP
backtracking, which while helful to gene expression seems also to prevent
errors in chromosome replication by reducing collisions between the DNA and RNA
polymerases \cite{dutta_linking_2011}. This shows shows a new way in which gene
expression is linked to genome duplication, hinting that we are still only
scratching the surface of the complexity of cell biology.

In order for the ribosome and the polymerase to carry out gene expression in a
coordinated fashion, it is vital that, once a transcript has been formed, the
ribosome manages to initiate translation efficiently.

XXX what are you trying to achieve here? It doesn't seem like this is one
story.

\subsubsection{Translation initiation}

We have previously covered transcription initiation, which involved the
5-protein complex RNA polymerase and the $\sigma$ unit. Translation initiation
is more complex than transcription initiation. Partly this is because the
ribosome is a much larger molecule than RNAP. The ribosome consist of two
subunits called S50 and S30, which when together are called the S70 complex.
The S symbolizes the svedberg unit, a nonlinear measure of the rate of
sedimentation of these molecules. All in all, the ribosome is made up from over
50 proteins, ten times a many as RNAp. In addition, it consists of over 4500
nucleotides of ribosomal RNA \cite{laursen_initiation_2005}, which in
\textit{E. coli} amounts to two thirds of the ribosome. Since the catalytic
activity of the ribosome (amino acid bond formation) is actually carried out by
the molecule's RNA fraction \cite{steitz_rna_2003}, the ribosome is not so much
a protein enzyme as an RNA enzyme, a ribozyme.

\subsubsection{Binding of S30 to the Shine-Dalgarno sequence}

A central ribosomal RNA (rRNA) in translation initiation is the 16S RNA which
is part of the S30 subunit. The sequence in the 3' end of this RNA is highly
conserved across bacteria and archea \cite{nakagawa_dynamic_2010}. It was shown
by Shine and Dalgarno in 1974 that the last nine bases of the 16S RNA in 
\textit{E. coli} are ACCUCCUUA \cite{shine_3-terminal_1974}. They suggested
that this sequence could bind to a conserved complementary motif in messenger
RNAs 5' UTR regions; this mRNA motif later came to be known as the
Shine-Dalgarno sequence. The Shine-Dalgarno sequence is often given as GGAGGA,
but the core motif is GGAG. It is found in most bacterial mRNA usually at a
distance of 3 to 10 bases from the start codon \cite{chen_determination_1994-1}
and is in these mRNA necessary for translation initiation to occurr. The area
around the Shine-Dalgaro site is called the ribosome binding site (RBS) and
occupies roughly 30 nucleotides.

The binding of 16S RNA of the 30S ribosomal subunit during translation
initiation achieves two things. The first is that he S30 subunit becomes
anchored to the transcript, and the second is that the start codon (often AUG)
becomes aligned to the peptidyl (P) site on S30 from where peptide synthesis
occurs. In the last sense, the Shine-Dalgarno (SD) sequence acts for the
ribosome like the promoter for the RNA Polymerase; both direct these
macromolecules to the site of \textit{de novo} synthesis.

It is in the nature of the 16S-SD RNA-DNA bond that the Shine Dalgarno
sequence must be unsequestered for the 16S to be able to base pair. Two possible
ways of sequestering the SD sequence is by protein binding and by RNA-RNA bonds
either from \textit{trans} elements like microRNA or from \textit{cis} elements
which are SD-complementary RNA on the transcript itself. Here, we focus on how
\textit{cis}-mediated sequestering of the SD site.

\subsubsection{RNA folding}
The basis for RNA folding is that RNA, as DNA, can basepair with itself; C
pairs with G and A pairs with U. As the nascent RNA emerges from the RNA exit
channel of the polymerase it is immediately free to fold into an energetically
favorable confirmation.

A key question in co-transcriptional RNA folding has been how comparable the
time scales for RNA folding and RNA synthesis are
\cite{de_smit_translational_2003-1}. Whether or not RNA folds
co-transcriptionally depends on the timescale for folding being shorter than
that of transcript production. For bacteria, nucleotides are incorporated at a
rate of between 20 and 80 ms per nucleotide, while formation and dissociation
of semi-stable helices occur on the 10 to 100 $\mu$s timescale
\cite{isambert_jerky_2009}. In other words, for each nucleotide produced, there
is on average time for 1000 refolding events. It should be noted, the time
needed for the spontaneous refolding of an RNA structure depends on the binding
strength of that structure. The cell has evolved mechanisms to either ensure or
prevent certain transient secondary structures during the folding pathway. The
purpose of this is to ensure that a desired final secondary (and in the case of
some RNA, tertiary) state is reached \cite{pan_rna_2006-1}.

\subsubsection{RNA secondary structures in the ribosome binding site}
It was early on showed that translation rates were stronly affected by
secondary structures at the RBS; changes to the secondary structures were found
to affect translation rates by a factor of over 500
\cite{de_smit_secondary_1990}. In particular, a non-linear relationship between
the folding energy of structures in the RBS and the translation initiation
rate was found: below a certain limit, the translation initiation rate is almost
invariant with respect to the strength of the secondary structure; above that
limit, the relationship is exponential:
\begin{equation*}
	\text{Translation rate} \approx exp^{\Delta G_{\text{RBS}}}
\end{equation*}

Not only the Shine-Dalgarno sequence but the entire RBS sequence, including
sequences downstream the start codon \cite{seo_quantitative_2009} must in
general be unstructured for translation initiaton to occur. It is therefore not
surprising that lack of strong secondary structures is a hallmark of ribosome
binding sites in \textit{E coli} \cite{gu_universal_2010} genome wide. The
folding energy of the RBS can even be used to distinguish between active genes
and pseudogenes \cite{keller_reduced_2012}, presumably because pseudogenes are
no longer under the selection pressure to keep the RBS weakly structured.

It should be noted that weak structures do not imply the absence of structures.
In general, it can be expected that the ribosome must deal with structured
binding sites. To account for this, it was initially suggested that ribosomes
would bind only as the structues in the RBS spontaneously unfolded
\cite{de_smit_translational_1994}. However, it was later apprecated that the
time scale for RNA folding and unfolding are orders of magnitude faster than
the time scale for free ribosome binding. This led to the suggestion that
ribosomes could bind so called ribosome standby site close to the RBS where
they could slide into place once the secondary structure opened
\cite{de_smit_translational_2003-1}. This was subsequently supported by a study
that showed that that the ribosome together with translation initiation factors
(IF) can unwind secondary structures at the RBS presumably from ribosome
standby sites \cite{studer_unfolding_2006}. The exact mechanism or the actor of
RNA unfolding is however not clear.

\subsubsection{Modifying RBS secondary structures to increase gene expression}
Given that secondary structures in the RBS have such a marked impact on
translation rates, many studies have varied the RBS-sequence for the purpose of
tuning gene expression \cite{cebe_rapid_2006} \cite{park_design_2007}
\cite{berg_expression_2009}. For example, it is often necessary to modify the
RBS sequence of eukaryotic genes when trying to express them in a prokaryotic
host \cite{care_translation_2007}.

In practise when seeking to optimize the expression of a given gene, labs turn
to commercial providers of gene optimization, such as DNA 2.0 or GenScript.
However, there are published alternatives. The most prominent of these is the
RBS calculator \cite{salis_automated_2009}. This software takes into account
the match of the 16S RNA to the SD sequence, the distance from the SD sequence
to the start codon, the folding energy of any RBS structures, the start codon,
and the folding energy of the ribosome standby site. Of these variables, the
one that correlatest most strongly with gene expression is the folding energy
of the RBS structures, see \cite{salis_automated_2009} in the supplementary
materials. By considering these variables, the software will suggest
modifications to a given gene sequence to set gene expression to a specified
level.

\subsubsection{RNA secondary structures affect RNA stability}
RNA stability means how long it takes from an RNA is produced until it is
degraded. The more stable a transcript is, the more time ribosomes have to
translate it to produce protein, meaning that more stable transcripts lead to
more protein. Also, RNA degradation is the final way to stop protein
production, making it a way for the cell to swiftly control protein production.
For these reasons, RNA stability is carefully regulated in the cell.

There are two well-known ways in which RNA secondary structures affect RNA
stability. The first is by hairpin structures at the 5' and 3' ends. The
hairpin at the 5' end protects the triphosphate of the 5' nucleotide against
attack from the protein RppH which converts 5' triphosphates to 5'
monophosphates \cite{deana_bacterial_2008}. In turn, RNAs with a 5'
monophosphate are targets for RNAse E, an endonuclease which can initiate RNA
degradation \cite{mackie_ribonuclease_1998}. In this way, a 5' hairpin
indirectly protects the mRNA against endonucleolytic attack. On the other hand,
the 3' hairpin protects directly against degradation from 3' exonucleases,
which need RNA with an unstructured 3' end for their activity
\cite{rauhut_mrna_1999}.

The second way RNA structures affect RNA stability is more indirect. As
mentioned, secondary structures at the RBS affects the rate of translation
initiation. The rate of translation again affects the spacing between
translating ribosomes on the transcript. Since it was early observed that
translated RNA is protected from degradation, the idea was put forth that
reduced spacing between translating ribosomes protects against endonucleolytic
cleavage by steric blocking. It has however been shown in at least two cases
that ribosome binding, but not translation, confer transcript stability
\cite{wagner_efficient_1994} \cite{hambraeus_5_2002}, indicating that narrow
spacing between translating ribosomes is not necessary for increased stability.
Still, the precise mechanism for how ribosome binding prevents degradation is
not clear \cite{deana_lost_2005}.

\subsubsection{The role of codon bias in gene expression and cellular fitness}
During translation elongation the ribosome matches incoming amino acids on
transfer RNAs (tRNA) to codons (nucleotide triplets; eg, ATG) on the mRNA. Each
codon corresponds to an amino acid, and in this way the RNA sequence is being
translated into a protein sequence. Since there are 61 codons that bind tRNA
($4^3$ minus three stop codons) but only 20 amino acids, it is inevitable that
several codons are associated with the same amino acid; this fact has been
called the redundancy of the genetic code. This redundancy and the observation
that some iso-coding codons were preferred over others puzzled early workers on
the genetic code, and still today these observations remain partially
unanswered (see a great review on the topic by Plotkin and Kudla
\cite{plotkin_synonymous_2011}).

Codons that code for the same amino acid are called synonoymous codons, and the
preference of one synonymous codon over another is called codon usage bias.
Codon usage bias is a universal phenomenon as it is found throughout the
kingdom of life \cite{sharp_codon_1988}. In many species, codon bias is
especially strong in highly expressed genes, which has lead to the suggestion
that some codons are more ``efficient'' than others and that genes biased in the
direction of efficient codons are translated more efficiently
\cite{moriyama_gene_1998}. It was eventually shown that efficient codons are
those with the highest copy number of tRNA genes, giving an explanation to the
enigma \cite{reis_solving_2004, elf_selective_2003}. A commonly used measure
for codon bias is the codon adaptation index (CAI), which measures how similar
a gene's codon content is compared to the codons in highly expressed genes in
the same organism \cite{sharp_codon_1987}. A practical consequence of codon
bias is that when expressing for example a human gene in a bacterial host, it
is often necessary to change the codons in the gene to the preference of the
bacteria (called codon optimization) to achieve high expression levels
\cite{gustafsson_codon_2004}, because the preferred codons between humans and
bacteria are different.

Two recent publications have shed light on the effect that codon bias has on
the rate of translation. In the first, Kudla et al. made random synomymous
codons changes in the green fluorecent protein (GFP) gene to generate a
library of 154 GFP genes with on average 114 different codons each
\cite{kudla_coding-sequence_2009}. They found no correlation between CAI and
the expression level of the GFP variants, contrary to what is found in many
naturally occuring genes. On the other hand, CAI was highly correlated with
cell growth rates. What did correlate with expression levels was the RNA
folding free energy around the start codon; however, RNA folding energy did not
correlate with growth rates. The authors conclude from this that codon bias
exists in highly expressed genes not to optimize translation rates, but to
optimize overall cellular fitness, presumably by freeing up ribosomes which
would otherwise be pausing on highly expressed but codon-inefficient genes due
to lack of incoming charged tRNA \cite{kudla_coding-sequence_2009}.

In the second paper, Tuller et al. \cite{tuller_evolutionarily_2010} examined
the codon bias of 27 organisms from all three domains of life by using the tRNA
adaptation index (tAI). This measure ranks each codon with the copy number of
the associated tRNA in the genome. They found a species-wide trend that genes
tended to have inefficient codons in the early coding region (first 30-70 codons),
but efficient codons in mid and late coding regions. The early inefficient
codons were labeled a ramp. They attribute this to that reduced speed in early
translating ribosomes reduce collisions later on, therby increasing overall
translation efficiceny in the cell.

A possible criticism for the authors of the Tuller et al. paper is that the
selection for unstructured mRNA around the RBS reduces the evolutionary freedom
of the early coding region compared to later regions
\cite{plotkin_synonymous_2011}, which could partially explain the ramp effect.

Finally, there are other sources of codon bias. One is the specific order that
codons appear in \cite{cannarozzi_role_2010}. Another is the avoidance of
message-bearing motifs like the Shine-Dalgarno element. It was shown in
\textit{E. coli} that when Gly-Gly amino acid pairs are coded for in a gene,
the most common codon pair is GGC-GGC, which has the lowest possible affinity for
the anti-Shine Dalgarno sequence, while the least common pair was GGA-GGU,
which is an exact match to the Shine-Dalgarno sequence
\cite{li_anti-shine-dalgarno_2012}. In the same study it was shown that
Shine-Dalgarno sequences inside coding regions could cause rebinding of the
16SRNA during translation elongation and thereby cause translation pausing.
Curiously, this rebinding behavior is the same as found for the sigma factor
during transcription elongation by RNA polymerase.
