%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
Translation of an RNA transcript into protein happens in different ways between
eukaryotes and prokaryotes. In eukaryotes, a transcript destined for
translation must be co-transcriptionally modified in several ways before it can
be transported out of the nucleus to the waiting ribosomes in the cytoplasm.
More about this in the next chapter. For now we focus on translation in
bacteria.

Since bacteria do not have a nucleus the ribosome has access to the
transcript immediately after its 5' end leaves the RNA exit channel. It has
been shown that the initially translating ribosome follows closely and even
pushes the transcribing polymerase, causing them to have the same speed
\cite{proshkin_cooperation_2010}. This ribosome pushing can prevent RNAP
backtracking, which while helful to gene expression seems also to prevent
errors in chromosome replication by reducing collisions between the DNA and RNA
polymerases \cite{dutta_linking_2011}. This shows shows a new way in which
gene expression is linked to genome duplication, hinting that we are still only
scratching the surface of the complexity of cell biology.

In order for the ribosome and the polymerase to carry out gene expression in
a coordinated fashion, it is vital that, once a transcript has
been formed, the ribosome manages to initiate translation efficiently. In this
thesis, chapters X and Y deal with how translation initiation is affected by
the folding of RNA in the ribosome binding site (RBS). Here, we review those
aspects of translation initiation that are relevant for these chapters. We also
briefly mention translation elongation and termination.

\subsubsection{Translation initiation}

We have previously covered transcription initiation, which involved the
5-protein complex RNA polymerase and the $\sigma$ unit. Translation initiation
is more complex than transcription initiation. Partly this is because the
ribosome is a much larger molecule than RNAP. The ribosome consist of two
subunits called S50 and S30, which when together are called the S70 complex.
The S symbolizes the svedberg unit, a nonlinear measure of the rate of
sedimentation of these molecules. All in all, the ribosome is made up from over
50 proteins, ten times a many as RNAp. In addition, it consists of over 4500
nucleotides of ribosomal RNA \cite{laursen_initiation_2005}, which in
\textit{E. coli} amounts to two thirds of the ribosome. Since the catalytic
activity of the ribosome (amino acid bond formation) is actually carried out by
the molecule's RNA fraction \cite{steitz_rna_2003}, the ribosome is not so much
a protein enzyme as an RNA enzyme, a ribozyme.

A central ribosomal RNA (rRNA) in translation initiation is the 16S RNA which
is part of the S30 subunit. The sequence in the 3' end of this RNA is highly
conserved across bacteria and archea \cite{nakagawa_dynamic_2010}. It was shown
by Shine and Dalgarno in 1974 that the last nine bases of the 16S RNA in 
\textit{E. coli} are ACCUCCUUA \cite{shine_3-terminal_1974}. They suggested
that this sequence could bind to a complementary motif in messenger RNAs 5' UTR
regions. This motif later came to be known as the Shine-Dalgarno sequence. The
Shine-Dalgarno sequence is often given as GGAGGA, but the core motif is GGAG.
It is found in most bacterial mRNA usually at a distance of 3 to 10 bases from
the start codon \cite{chen_determination_1994-1} and is in these mRNA necessary
for translation initiation to occurr.

The binding of 16S RNA of the 30S ribosomal subunit during translation
initiation achieves two things. The first is that he S30 subunit becomes
anchored to the transcript, and the second is that the start codon (often AUG)
becomes aligned to the peptidyl (P) site on S30 from where peptide synthesis
occurs. In the last sense, the Shine-Dalgarno (SD) sequence acts for the
ribosome like the promoter for the RNA Polymerase; both direct these
macromolecules to the site of \textit{de novo} synthesis.

It is in the nature of the 16S-SD RNA-DNA bond that the Shine Dalgarno
sequence must be unsequestered for the 16S to be able to base pair. Two possible
ways of sequestering the SD sequence is by protein binding and by RNA-RNA bonds
either from \textit{trans} elements like microRNA or from \textit{cis} elements
which are SD-complementary RNA on the transcript itself. Here, we focus on how
\textit{cis}-mediated sequestering of the SD site.

\subsubsection{RNA folding}

TODO read up on this part.
