%\bibliographystyle{plain}
%\bibliography{/home/jorgsk/phdproject/bibtex/jorgsk}
Transcription is the synthesis of an RNA molecule from a DNA template and the
macromolecule that drives transcription is the RNA polymerase (RNAP). RNAP
consist of the \alpha 1, \alpha 2, \beta $\beta^{\prime}$, and \omega subunits,
weighs in at around 400 kDa, and is conserved in structure and function across
all kingdoms of life \cite{borukhov_rna_2008}.

Here, we will review the role of RNAP in the initiation, elongation, and
termination of transcription. Special care will be given to the topic of
abortive transcription initiation and the model for how RNAP moves along DNA.

\subsubsection{Promoter localization by RNAP requires the sigma factor}
Before it can start transcription, RNAP must first localize a transcription
start site (TSS). These sites are advertised by promoter elements, which
stretch from -80 to +20 relative to the TSS. RNAP cannot bind the promoters
directly, but must associate with a regulatory protein called the sigma
($\sigma$) factor; the sigma factor in turn only binds strongly to promoters
when together with RNAP. In \textit{E. coli} there are 7 types of sigma
factors, and each sigma factor controls the binding to a particular gene family
\cite{osterberg_regulation_2011}. For example $\sigma^{70}$ recognizes
promoters for housekeeping genes, while $\sigma^{32}$ recognizes promoters for
genes that are activated in case of heat shock. Most promoters are of the
$\sigma^{70}$ type, and we will here focus on this type of promoters.
The most prominent $\sigma$-binding elements in these promoters are the -35 and
-10 hexamers, but an AT rich UP element and the so called discriminator region
between -10 and the (TSS) can also regulate the binding affinity of
$\sigma^{70}$ for the promoter.

\subsubsection{Transcription initiation}
Once RNAP-$\sigma$ -- together called the holoenzyme --  has bound to a
promoter, transcription initiation may commence. First, $\sigma$ mediates the
melting of the double stranded DNA from -11 to +2 relative to the transcription
start site.

% nucleotide = base-sugar-phosphate (1 or 3)
% nucleoside = base-sugar
% BUT! this confusing terminology exists: ``nucleotide with 3 phosphates'' =
% nucleoside triphosphate.
Now that it is single stranded, DNA is ready to be used as a template. RNAP
will position itself so that free DNA base of the transcription start site (+1)
will be in the active site of the polymerase. Now, this DNA base is free to
base-pair with an incoming nucleoside triphosphate (NTP), forming the first RNA
nucleotide. In order to incorporate the second nucleotide, the active site must
be cleared of the first nucleotide and be positioned in the +2 position. This movement
for the sake of of incorporating a new nucleotide is called translocation. Once
RNAP has translocated, the nucleotide corresponding to the +2 DNA base can
bind. To bind the two nucleotides, RNAP catalyzes a phosphodiester bond from
the phosphate group of the +2 nucleotide to the ribose sugar of the +1
nucleotide, a reaction which releases pyrophosphate (PPi). Now RNAP must
translocate again to make space for the +3 nucleotide. At each translocation
step, one basepair of the downstream DNA bubble must open, and the nascent RNA
is moved toward the RNA exit channel in RNAP.

This cycle of translocation and nucleotide incorporation is goes on although
sigma is still bound to the promoter. Thus, instead of moving downstream with
each translocation step, RNAP must pull DNA into itself in what has been
labeled 'scrunching'. It has been suggested that that buildup of stress from
the scrunching process eventually leads to the dissociation of sigma from the
promoter and allows RNAP to escape the promoter to the elongation phase of
transcription \cite{revyakin_abortive_2006}. However, promoter escape is only
one possible pathway for the initially transcribing RNAP. Another pathway is
abortive initiation, where RNAP releases the scrunched DNA and nascent RNA and
falls back to the open complex formation.

\subsubsection{Abortive initiation}

\subsubsection{Transcription elongation}
After having achieved promoter escape, RNAP keeps incorporates incoming
nucleotides to the 5' end of the RNA molecule, thereby elongating it.

This is a mess: put the active site stuff somewhere else. Put the pausing and
backtracking here.

The location in RNAP where the incoming nucleotide is joined to the RNA is
called RNAP's active site. After a nucleotide has been incorporated, RNAP must
move one basepair downstream the DNA template to free up a space for the next
incoming substrate; this specific movement is called translocation. As RNAP
elongates, DNA is kept open in what is called the DNA bubble, which extends
roughly from -12 to +2 relative to the RNAP active site.

\subsubsection{Transcription termination}
Eventually, RNAP falls of DNA and releases its RNA product. Two distinct
mechanisms have been identified for the release of RNA from RNAP. In one, the Rho
protein binds a ribosome-free unstructured region of at least 97 nucleotides of
RNA and moves along RNA in the direction of the RNAP until they meet at RNAP
pause sites, and their interaction causes the release of both RNA and RNAP from
the DNA template \cite{ciampi_rho-dependent_2006}. The
Rho-independent, or intrinsic, mechanism of termination is through a strong
(GC-rich) RNA hairpin followed by a U-rich sequence on RNA. After the GC rich
sequence is free from the RNA exit channel it folds into a hairpin, and
RNAP proceeds to transcribe the U-rich sequence which gives a weak RNA-DNA
hybrid. Interactions between RNAP and the hairpin together with the weak hybrid
then causes the RNA to be released \cite{nudler_transcription_2002}.

In both cases, once RNA has been released, the affinity of RNAP to DNA is
greatly reduced and RNAP falls off. It is then free to associate with a
sigma factor and begin transcription again.

\subsubsection{Nucleotide incorporation and translocation}
Nucleotide incorporation happens in the post-translocated state of RNAP, where
the active site, or i+1, is free for basepairing with the incoming nucleoside
triphosphate (NTP). When the appropriate NTP has arrived through the secondary
channel and bound the DNA template in the active site, RNAP joins the incoming
substrate to the 3' end of the RNA with a phosphodiester bond, which releases
PPi. This puts RNAP in the pre-translocated state again, and RNAP must now
translocate once more to free up the active site for nucleotide synthesis. This
cycle of substrate binding, phosphodiester bond formation, and translocation,
constitute the forward motion of RNAP on DNA.

\subsubsection{Conceptual model for RNAP movement on DNA}
The movement of RNAP on DNA is thought to occur by a so called Brownian ratchet
mechanism. In this model, the backward movement of RNAP is structurally
prevented by the RNAP F bridge \cite{_ratchet_2005}, while the forward motion
is free to happen by brownian motion and as recently shown a translocation bias
\cite{hein_rna_2011}.

Even though the backward motion is prevented, it can occasionally happen,
leading to backtracked elongation complex. When this occurs, the 3' end of the
RNA product ends up in the RNAP secondary channel, blocking the active site
from binding incoming substrate. To rescue backtracked complexes, the GreA and
GreB transcription factors can bind in the secondary channel and stimulate the
intrinsic RNA cleavage activity of RNAP at the active site. When RNAP cleaves
the RNA, the backtracked RNA is end is released and a new 3' end is formed at
the active site, making RNAP again translocation-competent
\cite{toulme_grea_2000}.

The cause for the backward motion and other types of RNAP pausing during
transcription has been laid on the free energy balance between the RNA-DNA
hybrid, the DNA-DNA bubble, and an unknown stabilizing effect by
RNAP\cite{greive_thinking_2005}. By assuming the reaction steps that involve
each of these free energy values are in equilibrium
\cite{greive_thinking_2005}, the total free energy of the elongation complex
can be written as 
\begin{equation}
	\Delta G_{\text{Total}} = \Delta G_{\text{RNA-DNA}} + \Delta
	G_{\text{DNA-DNA}} + \Delta G_{\text{RNAP}}
	\label{eq:rnap_energy_balance}
\end{equation}
Depending on this energy term, it is proposed that RNAP will either move
forward, backtrack, pause, or even terminate.

\subsubsection{Computational models of transcription}
Since the first two terms of equation \eqref{eq:rnap_energy_balance} can be
calculated from published energy tables \cite{wu_temperature_2002}
\cite{santalucia_thermodynamics_2004}, several kinetic and thermodynamic models
of transcription elongation have been published
\cite{tadigotla_thermodynamic_2006-1} \cite{bai_sequence-dependent_2004}
\cite{guajardo_model_1997}. What they have in common is that in some form they
incorporate the terms from \eqref{eq:rnap_energy_balance} (in the case of
Tadigotla et. al the free energy from nascent RNA secondary structures close
to the RNA exit channel is used as well). When compared with measurement data,
however, these models leave much to be desired, showing that there is more to
RNAP processivity than the parameters in \eqref{eq:rnap_energy_balance}.

\subsubsection{A computational model of transcription initiation}
The difference between transcription initiation and transcription elongation is
i) that during initiation the RNA-DNA hybrid lacks its full length until +8/9,
ii) that the DNA-DNA bubble is scrunched and pulled into RNAP, and iii) that
RNAP is bound to $\sigma$ which again is bound to the promoter. The consequence
of these difference is that the TIC has a reaction pathway that doesn't exist
during elongation: abortive initiation.

Based on equation \eqref{eq:rnap_energy_balance} and the above differences, a
model for transcription initiation was previously published
\cite{xue_kinetic_2008}. This model attempts to cover explicitly both the
abortive process and the actual promoter escape event. Although these authors
manage to reproduce well the maximum size of abortive transcript and the
abortive to productive ration of the N25 promoter, they do not explain well the
abortive probabilities. This study investigates only the N25, N25anti and T7A1
promoters, even though it was published after a larger dataset was available
\cite{hsu_initial_2006}. Further, the T7A1 promoter varies not only in the ITS
sequence with the other two, but also in the core promoter sequence. Variation
in the core promoter has been shown to affect rates of abortive initiation
\cite{vo_vitro_2003-1}, yet these authors have not taken this variation into
account.
