%\bibliographystyle{plain}
%\bibliography{/home/jorgsk/phdproject/bibtex/jorgsk}
Transcription is the synthesis of an RNA molecule from a DNA template, and the
macromolecule that performs transcription is the DNA-dependent RNA polymerase
(RNAP). RNAP consists of the $\beta$, $\beta^{\prime}$, $\omega$, and two
$\alpha$ subunits, weighs in at around 400 kDa, and is conserved in structure
and function across all kingdoms of life \cite{borukhov_rna_2008}.

Here, we will review the role of RNAP in the initiation, elongation, and
termination of transcription. Special care will be given to the topic of
abortive transcription initiation and the model for how RNAP moves along DNA.

\subsubsection{Promoter localization}
Before RNAP can start transcription it must localize a transcription start site
(TSS). These sites are advertised by promoter elements, which stretch from -80
to +20 relative to the TSS. RNAP cannot bind the promoters directly, but must
associate with a regulatory protein called the sigma ($\sigma$) factor; the
sigma factor in turn only binds strongly to promoters when together with RNAP
\cite{paget_70_2003}. In \textit{E. coli} there are 7 types of sigma factors,
and each sigma factor controls the binding to a particular gene family
\cite{osterberg_regulation_2011}. For example $\sigma^{70}$ recognizes
promoters for housekeeping genes, while $\sigma^{32}$ recognizes promoters for
genes that are activated in case of heat shock. Most promoters are of the
$\sigma^{70}$ type, and we will here focus on this type of promoters. The most
prominent $\sigma$-binding elements in these promoters are the -35 and -10
hexamers, but an AT rich UP element and the so called discriminator region
between -10 and the (TSS) can also affect the binding affinity of $\sigma^{70}$
for the promoter \cite{ross_third_1993} \cite{haugen_fine_2008}.

\subsubsection{Transcription initiation}
% nucleotide = base-sugar-phosphate (1 or 3)
% nucleoside = base-sugar
% BUT! this confusing terminology exists: ``nucleotide with 3 phosphates'' =
% nucleoside triphosphate.
Once RNAP-$\sigma$ -- together called the holoenzyme --  has bound to a
promoter, transcription initiation may commence. First, $\sigma$ mediates the
melting of the double stranded DNA from -11 to +2 relative to the transcription
start site. Now that it is single stranded, DNA is ready to be used as a
template. RNAP will position itself so that free DNA base of the transcription
start site (+1) will be in the active site of the polymerase. Now, this DNA
base is free to base-pair with an incoming nucleoside triphosphate (NTP),
forming the first RNA nucleotide. In order to incorporate the second
nucleotide, the active site must be cleared of the first nucleotide and be
positioned in the +2 position. This movement for the purpose of of incorporating a
new nucleotide is called translocation. Once RNAP has translocated, the
nucleotide corresponding to the +2 DNA base can bind. To join the two
nucleotides, RNAP catalyzes a phosphodiester bond from the phosphate group of
the +2 nucleotide to the ribose sugar of the +1 nucleotide, a reaction which
releases pyrophosphate (PPi). Subsequently RNAP must translocate again to make
space for the +3 nucleotide. At each translocation step, one basepair of the
downstream DNA bubble must open, and the nascent RNA is moved toward the RNA
exit channel \cite{o_maoileidigh_unified_2011}.

This cycle of translocation and nucleotide incorporation goes on although sigma
is still bound to the promoter. This has the consequence that instead of being
free to move downstream with each translocation step to expose more DNA
template, RNAP must pull DNA into itself in what has been labeled 'scrunching'.
It has been suggested that that buildup of stress from the scrunching process
eventually leads to the dissociation of sigma from the promoter and allows RNAP
to escape the promoter to the elongation phase of transcription
\cite{revyakin_abortive_2006}.

\subsubsection{Abortive initiation}
Promoter escape is only one out of two possible outcomes for a transcription
initiation attempt. If the promoter-$\sigma$ contacts are not broken by around
the +16 position, most initiation complexes will have aborted the escape
attempt and released the nascent RNA into solution
\cite{lilian_m_promoter_2002}. The aborted transcripts are generally no longer
than 15 nucleotides, but lengths up to 20 have been observed
\cite{chander_alternate_2007}. In \textit{in vitro} experiments, abortive
initiation is the rule rather than the exception, with some promoter variants
having a ratio of aborted to successful transcription initiation attempts of
more than 300 \cite{hsu_initial_2006}. This indicates that RNAP may on average
spend considerable time in abortive cycling before achieving promoter escape.

When comparing abortive initiation at different promoters, it became clear that
both the lengths of the abortive transcripts and the extent of abortive cycling
were promoter-specific. Although elements in the core
promoter region affected abortive initiation \cite{vo_vitro_2003}, the most
striking observation was that sequence composition of the +1 to +20 initially
transcribed sequence (ITS) had a strong effect on the lengths of the
abortive transcripts \cite{hsu_vitro_2003}. A comprehensive library of 43 ITS
variants with the N25 core promoter showed that there was a more than 20-fold
difference in the rate of productive synthesis depending on the ITS composition
\cite{hsu_initial_2006}. 

For a while it was speculated that abortive initiation was an \textit{in vitro}
phenomenon. Recently though small abortive RNAs were also found \textit{in
vivo} \cite{goldman_direct_2009}. It was also for a long time not know if the
short abortive transcripts had any cellular function or if they were merely
artefacts of the transcription initiation process. Yet in the last two years two
studies have shown two different uses of these short transcripts. In one,
aborted transcripts from the $\phi$10 promoter were found to deactivate the
T$\phi$ transcriptional terminator hairpin \cite{lee_tiny_2010}. In the other,
short abortive products of 2 to 4 nucleotides in length were found to act as
primers for the RNA polymerase \textit{in vivo} \cite{goldman_nanornas_2011};
previously, it was not known if RNAP, unlike the DNA polymerase, could use
primers \textit{in vivo}.

So far, it has not been shown if abortive initiation is rate-limiting for
natural promoters \textit{in vivo}. \textit{In vitro} and \textit{in vivo}
experiments have however shown that the transcription factors GreA and GreB
greatly reduce the amount of abortive initiation from an artificial promoter
\cite{hsu_escherichia_1995}, indicating that these transcription factors could
act in the cell to reduce the promoter-specific variability in abortive
initiation. GreA and GreB act by stimulating the cleavage of RNA in the active
site of backtracked RNAP \cite{toulme_grea_2000}, which are RNAP that have the
3' end of the nascent RNA protruding out the NTP entry channel, thus putting
the RNA out of alignment with the active site and blocking further RNA
synthesis. It is thus possible that initially transcribing complexes on the
path to abortive initiation \textit{in vivo} require RNA cleavage realign the
RNA 3' OH group at the active site. In support of an \textit{in vivo} role for
GreA in mitigating abortive initiation, it was found that GreA resolves
promoter-proximal stalling of RNAP \cite{kusuya_transcription_2011}. However,
more work is needed to confirm the precise role that abortive initiation plays
in gene expression.

\subsubsection{Transcription elongation}
Once the sigma-promoter bonds are broken, RNAP is free to elongate onto the
downstream DNA. After the nascent transcript reaches a length of more than 15
nucleotides, it should have displaced both the R3.2 linker and region 4 of
$\sigma^{70}$ \cite{mekler_structural_2002} \cite{nickels_interaction_2005},
weakening the bonds between RNAP and sigma. It is thought that sigma release is
stochastic after promoter escape, and several studies have found that if sigma
is still attached to RNAP in elongation it can rebind to promoter-like
elements, causing transcriptional pausing \cite{ring_function_1996},
\cite{kapanidis_retention_2005}, \cite{raffaelle_holoenzyme_2005}.

Transcription elongation happens with great processivity: RNAP can accurately
transcribe tens of thousands of nucleotides without falling off. This stability
has been attributed to the $\approx$ 9 nt RNA-DNA hybrid
\cite{nudler_rna-dna_1997}. Still, elongation does not happen at an even pace.
RNAP will reproducibly pause or backtrack at certain sites
\cite{herbert_sequence-resolved_2006} , and sometimes this pausing or
backtracking has a regulatory function \cite{landick_r_regulatory_2006}.

\subsubsection{Transcription termination}
Eventually, RNAP falls of DNA and releases its RNA product. Two distinct
mechanisms have been identified for the release of RNA from RNAP. In one, the Rho
protein binds a ribosome-free unstructured region of at least 97 nucleotides of
RNA and moves along RNA in the direction of the RNAP until they meet at RNAP
pause sites, and their interaction causes the release of both RNA and RNAP from
the DNA template \cite{ciampi_rho-dependent_2006}. The other, Rho-independent
mechanism of termination is by the formation of a strong (GC-rich) RNA hairpin
right outside the RNA exit channel. If this hairpin is followed by a U-rich
sequence on RNA which destabilizes the RNA-DNA hybrid, interactions between the
hairpin and RNAP cause RNAP to release its hold on the RNA, although the
details of the process are not clear \cite{nudler_transcription_2002}.

In both cases, once RNA has been released, the affinity of RNAP to DNA is
greatly reduced and RNAP itself disengages DNA. It is then free to associate
with a sigma factor and begin transcription again.
