%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
\section{Summary of the paper}
The primary aim of the publication by Kucharova et al., attached in the
Appendix on page \pageref{vero_paper}, was to express in \textit{E. coli} a
variant of the human interferon gene called \textit{inf-$\alpha$2b}. The
protein product INF-$\alpha$2b is of pharmaceutical interest as a drug for
treating hepatitis C \cite{manns_peginterferon_2001}. The expression of a
foreign gene in a host organism is called heterologous expression, and carries
with it many challenges \cite{gustafsson_codon_2004}. Since heterologous
expression of INF-$\alpha$2b is necessary to produce the drug in large
quantities, and \textit{E. coli} is a much used expression host, it is
important to investigate the limitations and mechanisms of
\textit{inf-$\alpha$2b} expression in \textit{E. coli}.

Kucharova et al.\ reports that the \textit{inf-$\alpha$2b} gene is not
expressed in \textit{E. coli} in its native form. A codon optimized version of
\textit{inf-$\alpha$2b} was obtained to investigate if the non-native codon
usage in the gene was behind the lack of expression. That is, the native human
codons were replaced with codons that are in high usage in \textit{E. coli}.
However, in spite of of codon optimization no detectable expression of
transcript or protein could be found. Protein product was only detected when a
5\ppp fusion tag was added to the \textit{inf-$\alpha$2b} gene. This indicated
that events occurring in the 5\ppp region of the gene is involved in regulating
the switch between expression and nonexpresssion of the \textit{inf-$\alpha$2b}
gene.

In their paper, Kucharova et al.\ suggest that translation initiation of the
\textit{inf-$\alpha$2b} transcript is a possible reason why the gene is not
expressed. Further, they note that codon usage in first translated codons are
also known to impact gene expression. That prompts them to investigate variants
of \textit{inf-$\alpha$2b} that make synonymous codon substitutions to reduce
RNA secondary structures around the ribosome binding site (RBS). Some of these
variants result in detectable transcript levels for \textit{inf-$\alpha$2b}.
However, the increase in transcript level is not followed by an increase in
protein level. This suggests that further barriers than ribosome binding lie
behind the poor expression of \textit{inf-$\alpha$2b}. Having established this,
Kucharova et al.\ move on to investigate the effect of different 5\ppp terminal
fusion peptides. They show that that several short versions of the celB fusion
peptide increase expression of the \textit{inf-$\alpha$2b} gene and several
other genes. Finally, they improve the expression level with the celB leader by
screening a random mutagenesis library of celB around the RBS. In conclusion,
these fusion peptides probably facilitate translation initiation both by having
favorable secondary structure and also by some downstream effect that could not
be achieved only by alleviating RNA secondary structures at the RBS.

\section{Bioinformatic contribution}
As described in the paper by Kucharova et al.\ (see the Materials and Methods
section of the paper) the bioinformatic contribution to the paper was primarily
the construction an \textit{in silico} library of \textit{inf-$\alpha$2b}
variants, and the screening of this library with respect to: i) RNA free energy
around the RBS and ii) a score for the codon usage of the first 8 codon after
the start codon. See Figure \ref{fig:ensemble} for a graphical representation
of the library in terms of the free energy and codon score of each variant,
where the variants that were experimentally tested are indicated.

\begin{figure}[b]
	\begin{center}
		\includegraphics[scale=0.3]{figures/celb/ensemble.pdf}
	\end{center}
	\caption{\textit{in silico} library with 7680 synonymous variants of
	\textit{inf-$\alpha$2b}. The dark red circle at the top is the wild type codon
	optimized \textit{inf-$\alpha$2b}. In red are shown the variants that were
	selected for experimental testing.}
	\label{fig:ensemble}
\end{figure}

\subsection{Bioinformatic implementation}
Here, we will outline, in pseudo code, the implementation of the source code
that was used to produce and process the \textit{in silico}
\textit{inf-$\alpha$2b} library. The actual implementation is available as
outlined in the Appendix on page \pageref{source_code}.

\inputminted[fontsize=\small]{python}{pseudo_code/pseudo_code.py}
