%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
The target of this thesis was to use computational tools in combination with
wet-lab experiments to study regulation of gene expression.

In chapter \ref{chap:initiation_paper}, we found that the sequence of the ITS
modulates the promoter escape efficiency through its effect on translocation.
To do that, we first used a thermodynamic model of translocation that took into
account the free energy change of the DNA bubble, the RNA-DNA hybrid, as well
as free energy change associated with the sequence of the RNA 3\ppp
dinucleotide. Then we used these calculations to perform follow-up experiments
which confirmed the initial analysis. This study laid to rest the 25-year case
of why the initial transcribed sequence modulates promoter strength and
promoter escape efficiency. In addition, contrary to what was previously
assumed, we found no role for the free energy of the scrunched DNA bubble in
modulating promoter escape efficiency.

In chapter \ref{chap:celB}, we found that decreasing the RNA secondary
structure around the ribosome binding site led to increased transcript
expression of the \textit{inf-$\alpha$2b} gene in \textit{E. coli}, likely as a
result of increased RNA stability through ribosome binding. We did this by
\textit{in silico} screening a library of variants of \textit{inf-$\alpha$2b}
that had synonymous mutations in the first 9 codons, and testing those variants
whose synonymous mutations led to decreased secondary structures in the
ribosome binding site. This shows that heterologous gene expression can be
limited by secondary structures around the translation start site.

In chapter \ref{chap:polyA}, we classified evidence of polyadenylation in
different cell compartments for 12 human cell lines. We found evidence of
polyadenylation of poly(A)- RNA in intergenic and intronic regions. This might
be evidence of the recently discovered nuclear degradation-related
polyadenylation in human cells. We found these signals by building Utail, a
pipeline for screening RNA-seq reads for poly(A) stretches, trimming the reads,
and remapping the trimmed read to the genome to find the origin of the poly(A)
stretch. These results show the importance of studying the transcriptome
separately for different cell compartments, and suggest that RNA with short
poly(A) tails are abundant in the nucleus.

We conclude firstly that the findings in this thesis highlight different ways
in which gene sequences are involved in regulation of gene expression, and
secondly that the results in this thesis show how bioinformatics can fruitfully
be used together with experimental biology to explore the reationship between
gene sequence and gene expression.
