Gene expression -- the synthesis of RNA and protein -- requires most of the
cell's energy and is a highly regulated process at all its levels. In this
thesis, three studies are presented which focus on the regulation of three
different levels of gene expression. One is centered on the regulation of
transcription initiation; the other is about translation initiation; while the
third looks at RNA post-transcriptional processing. The studies are integrated
in the way that they illustrate how the DNA sequence directly affects
regulation at these different levels. They rely on different experimental data,
however, and require different theoretical techniques of analysis.

The study on transcription initiation is about how the RNA polymerase (RNAP)
takes the step from promoter binding to promoter escape. Before reaching
promoter escape, RNAP may undergo abortive initiation. This involves a release
of the nascent RNA transcript, although RNAP still remains bound to the
promoter, ready to begin RNA synthesis \textit{de novo} after RNA release. At
many promoters, RNAP undergoes repeated abortive initiation, known as abortive
cycling, before promoter escape is reached. The DNA sequence composition of the
20 first transcribed basepairs was known to affect the efficiency of promoter
escape, presumably by affecting the probabilities of abortive initiation, but
it was not known how this happened. Using a thermodynamic model of
transcription initiation, we have shown that the manner in which RNA polymerase
translocates during initial transcription explains the observed variation in
promoter escape efficiency on the N25 promoter. The key variable for describing
translocation for RNA polymerase during initiation was the sequence of the RNA
3\ppp dinucleotide, not the free energy of the DNA bubble, has had been
postulated by others. We proceed to verify our findings with follow-up
experiments, in which we used our thermodynamic model to construct N25 promoter
variants with predicted variation in promoter escape efficiencies. The
experiments agreed with the predictions, showing that translocation during
initial RNA synthesis is the major determinant of promoter escape efficiency.

The study on translation initiation focuses on the problem of heterologous
expression of the human \textit{inf-$\alpha$2b} gene in a bacterial host. A
known bottleneck for heterologous gene expression is the binding of the
ribosome binding to the translation initiation region of an mRNA. Therefore, we
introduced silent mutations in the first 9 codons of \textit{inf-$\alpha$2b}
that reduce the free energy of RNA secondary structures at the ribosome binding
site. This greatly increased the amount of \textit{inf-$\alpha$2b} transcript
for many of the variants, showing that translation initiation is likely a
strong barrier for the expression of this gene.

In the third study, we focus on the process of 3\ppp cleavage and
polyadenylation of eukaryote RNAs. Cleavage and polyadenylation make up a part
of pre-RNA processing of many eukaryote RNAS, and is required for mRNA
stability and transport from the nucleus to the cytoplasm. To study 3\ppp
cleavage and polyadenylation, we analyzed RNA sequencing data to identify
polyadenylation sites in transcripts from different cell compartments across 12
human cell lines. We found over 160.000 polyadenylation sites in all cell
lines, only 80\% of which were previously annotated. In addition we found an
unexpected enrichment of polyadenylation sites in nuclear poly(A)$-$ RNA in
intronic regions. These sites may be signals of polyadenylation-related nuclear
degradation.

In this thesis to different approaches have been used to study gene regulation;
a gene-centric approach for transcription and translation initiation, and a
genome-centric approach for 3\ppp cleavage and polyadenylation. This has
resulted in different types of research questions being asked, different
computational challenges, and different sorts of results, with the gene-centric
results being more mechanism oriented, and the genome-centric being more
general.

The main conclusion of the thesis is that free energy calculations, especially
when used in combination with traditional molecular biology techniques, are
efficient and useful tools for investigating the relationship between gene
sequence and gene expression for RNA polymerase-dependent transcription
initiation and ribosome-dependent translation initiation.
