%\addbibresource{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
%\bibliography{/home/jorgsk/phdproject/bibtex/jorgsk.bib}
A computational model of transcription can be used alongside wet-lab
experiments to improve our knowledge of how RNA synthesis occurs. Such a model
has two main uses: the first is to evaluate our existing conceptual models of
transcription by writing that conceptual model in a mathematical language and
comparing the resulting computational model with the available experimental
data. If the mathematical description cannot fit the data, it is a sign that
the conceptual model may be incomplete. Another use of such a model is to
experiment with the computational model: if a modification of the
computational model gives an improved fit to experimental data, it could be
worthwhile to investigate if that modification can be validated through
wet-lab experiments.

\subsubsection{The nucleotide addition cycle and the equilibrium assumption of
translocation}
The core of the computational models of transcription is a mathematical
description of the nucleotide addition cycle (NAC). In a two-step description,
the NAC consists of a reversible translocation step, where the RNAP active site
is made available for NTP binding, and a synthesis step, where an incoming
bound NTP is added to the 3\ppp end of the growing RNA chain (Figure
\ref{fig:nac_1}). While the NTP binding-and-synthesis step is reversible
through pyrophosohorolysis, this is highly unfavorable under typical
experimental conditions \cite{buc_chapter_2009}.
\begin{figure}[htb]
	\begin{center}
		\includegraphics[scale=0.4]{illustrations/sysbio/nucleotide_addition_cycle_1.pdf}
	\end{center}
	\caption{The nucleotide incorporation cycle proceeds through
	translocation and NTP incorporation. Before NTP binding, RNAP is in the
	pre-translocated state. RNAP then translocates, thereby entering the
	post-translocated state. In the post-translocated state, NTP can bind.
    After the incoming NTP has bound and become incorporated onto the
    3\protect\ppp end of the RNA, RNAP is once again in the pre-translocated
    state.}
	\label{fig:nac_1}
\end{figure}

For most published computational models of transcription, a key assumption
about the NAC is that the reversible translocation step attains equilibrium
before the NTP synthesis step \cite{greive_thinking_2005,
bai_mechanochemical_2007, guajardo_model_1997}. While it is difficult to
conclusively demonstrate that the equilibrium assumption always holds true, it
does facilitate the description of translocation with an equilibrium equation:
\begin{equation}
	Keq = e^{-\frac{\Delta G_{\text{RNA-DNA}} + \Delta
	G_{\text{DNA-DNA}} + \Delta G_{\text{RNAP}}}{RT}}.
	\label{eq:rnap_energy_balance}
\end{equation}
This equation contains the terms currently believed to contribute energetically
to translocation. These are the free energy change of the RNA-DNA hybrid
($\Delta G_{\text{RNA-DNA}}$), the DNA-DNA transcription bubble ($\Delta
G_{\text{DNA-DNA}}$), as well as a term for non-specific interactions between
RNAP, DNA, and RNA ($\Delta G_{\text{RNAP}}$) \cite{greive_thinking_2005}. The
latter term is not known and has previously been set to a
constant value \cite{tadigotla_thermodynamic_2006,
bai_sequence-dependent_2004}.

By calculating the equilibrium constant of translocation, one finds the
equilibrium balance between the pre-translocated and post-translocated states.
This constant can in the next step be used in a model of transcription to
calculate the probabilities of NTP binding, pausing, or backtracking
\cite{greive_thinking_2005, bai_mechanochemical_2007, guajardo_model_1997}.
Thus, the calculation of translocation is central to the description of
transcription.

\begin{figure}[htb]
	\begin{center}
		\includegraphics[scale=0.4]{illustrations/sysbio/nucleotide_addition_cycle_2.pdf}
	\end{center}
	\caption{The equilibrium constant of translocation is a simple
	model of the movement of RNAP along DNA during transcription.}
	\label{fig:nac_2}
\end{figure}

This conceptual model of RNAP movement as an equilibrium reaction seems
orderly: one needs only to calculate the change in free energy of $\Delta
G_{\text{DNA-DNA}}$ and $\Delta G_{\text{RNA-DNA}}$ from available energy
tables \cite{wu_temperature_2002, santalucia_thermodynamics_2004} to find out
if RNAP will move forward, backtrack, pause, or terminate at any given location
on DNA. However, it is not known how large the contribution of the $\Delta
G_{\text{RNAP}}$ term to equation \eqref{eq:rnap_energy_balance} is, or if
this term is sequence dependent. One way to investigate the veracity of
the conceptual model of RNAP movement is by using the equilibrium equation for
computational modelling of transcription. This enables a more rigorous testing
of the conceptual model by comparing the computational model results with
experimental data.

\subsubsection{Computational models of transcription elongation}
Several kinetic and thermodynamic models of transcription elongation have been
published \cite{tadigotla_thermodynamic_2006, bai_sequence-dependent_2004,
guajardo_model_1997, yager_thermodynamic_1991}. What they have in common is
that in some form they incorporate the terms from
\eqref{eq:rnap_energy_balance} and calculate the $\Delta G_{\text{RNA-DNA}}$
and $\Delta G_{\text{DNA-DNA}}$ terms from published tables of these values
for different dinucleotide combinations. In the case of Tadigotla et al.\
\cite{tadigotla_thermodynamic_2006} the free energy from the nascent RNA
secondary structure close to the RNA exit channel is used as well; this
calculation was used to model the effect RNA structures outside RNAP have in
preventing backtracking of RNAP \cite{zamft_nascent_2012}.

The early computational models of transcription had only partial predictive
power. For example, Tadigotla et al.\ \cite{tadigotla_thermodynamic_2006}
predict pause sites during transcription elongation. While their best optimized
model manages to identify 84\% of pauses, only 68 \% of their predicted pause
sites overlapped with experimentally identified pause sites, indicating that
there were a number of false positives. The model of Bai et al.\
\cite{bai_mechanochemical_2007} was published without any statistical
measures, making it difficult to interpret. Instead, Tadigotla et al.\
implemented the algorithm from Bai et al.\ and found that the Bai et al.\
model performed only little better than random for detecting pausing
\cite{tadigotla_thermodynamic_2006}. The performance of these models signalled
that more work was needed to reach a mature and accurate model for
transcription.

Recently, Maoil√©idigh et al.\ published a model of transcription elongation
which fitted well several transcription parameters that had previously been
measured by single-molecule experiments \cite{o_maoileidigh_unified_2011}. To
adapt their model to the results, however, they added to the model an
intermediary state between processive translocation and backtracking which has
not been observed experimentally \cite{o_maoileidigh_unified_2011}. Their
results suggests that translocation does not occur as an equilibrium reaction,
which contradicts the this previously made assumption. While this is the most
up to date model of transcription to date, this model still relies only on the
free energy change of the DNA bubble and RNA-DNA hybrid to calculate
translocation \cite{o_maoileidigh_unified_2011}. As will be discussed below, it
is now known that more terms than these influence this process, which
necessitate an additional free energy variable for the description of
translocation.

Recently, Hein et al.\ \cite{hein_rna_2011} showed that the sequence of the
3\ppp dinucleotide of the nascent RNA has a strong effect on translocation
rates.  Hein et al.\ found that if the 3\ppp nucleotide of the RNA was U, RNAP
had a much stronger preference for the pre-translocated state than if the 3\ppp
nucleotide was G. This finding was supported by Malinen et al.\
\cite{malinen_active_2012} who proposed that contacts between the RNA 3\ppp end
and the RNAP active site determine the preference for the pre-translocated or
the post-translocates states. The findings by Hein et al.\ and Malinen et al.\
have the potential to greatly improve on the existing computational uses of
translocation, since so far only free energy change of the RNA-DNA hybrid
and the DNA bubble have been the sole energy variables that drive translocation.

\subsubsection{A computational model of transcription initiation}
In addition to the above described models of transcription elongation, one
model bye Xue et al.\ \cite{xue_kinetic_2008} has previously been published for
transcription initiation.

For the sake of modeling, the key differences between transcription initiation
and transcription elongation is that during initiation i) the DNA-DNA
bubble grows with scrunching-translocation instead of being constant as for
elongation ii) the RNA-DNA hybrid lacks its full length until
the active site has reached +9/10, and iii) that RNA secondary structures
outside RNAP do not influence backtracking during initiation since these
structures cannot form until after promoter escape when a sufficient length of
RNA has emerged from the RNA exit channel. By incorporating these changes into
equation \eqref{eq:rnap_energy_balance}, one can model transcription initiation
in a similar manner as for elongation.

The model by Xue et al.\ takes as input the ITS of a promoter and predicts
measurable output such as the probability to abortively release nascent RNA at
a given position, and the ratio of abortive to successful initiation attempts.
The model is able to accurately predict the maximum size of abortive transcript
and the abortive to productive ratio of the N25 promoter. However, the model
does not mange to predict the abortive probabilities for different lengths of
the nascent RNA. Some criticism of this study is that only the data from the
N25, N25$_{\text{anti}}$ and T7A1 promoters were investigated, even though an
equivalent but more comprehensive dataset with 43 ITS variants was available
\cite{hsu_initial_2006}. Further, the T7A1 promoter has sequence variation in
the core promoter sequence compared to the other two, which only vary in the
ITS.  Variation in core promoter sequence has been shown to affect promoter
escape properties \cite{vo_<i>vitro</i>_2003}, yet the model does not take this
into account. This makes it difficult to compare the model output for the
analysis of the three different promoters. Finally, the model by Xue et al.\
was not used predictively, in the sense that it was not used to predict
abortive initiation properties of new DNA sequences, making it difficult to
conclude how much of the dynamics of transcription initiation it truly
captures.

In conclusion, the study by Xue et al.\ was the first quantitative model for
transcription initiation, and the model managed to predict some experimentally
measured parameters, although the lack of predictive usage of the model makes
it difficult to evaluate. Therefore, in the field of transcription initiation,
it remains to be published quantitative models with predictive power that
provide new insight into the process itself.
